samples/meetings-events/csharp/MeetingEvents/AppPackage/manifest.json

```diff
@@ -41,10 +41,28 @@
   ],
   "webApplicationInfo": {
     "id": "<<APP-ID>>",
-    "resource": "https://AnyString",
-    "applicationPermissions": [
-      "OnlineMeeting.ReadBasic.Chat",
-      "ChannelMeeting.ReadBasic.Group"
-    ]
+    "resource": "https://AnyString"
+  },
+  "authorization": {
+    "permissions": {
+      "resourceSpecific": [
+        {
+          "name": "OnlineMeeting.ReadBasic.Chat",
+          "type": "Application"
+        },
+        {
+          "name": "ChannelMeeting.ReadBasic.Group",
+          "type": "Application"
+        },
+        {
+          "name": "OnlineMeetingParticipant.Read.Chat",
+          "type": "Delegated"
+        },
+        {
+          "name": "ChannelMeetingParticipant.Read.Group",
+          "type": "Delegated"
+        }
```

Comment:
Not required, remove it.
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAy4AAABvCAYAAAAKX5T/AAAgAElEQVR4nO29f3hb1ZXv/ZV/xD6OlVhyIMhhBLaqwDR1gmGuUnxL7V6TxmnG3JK6gyBPuUMSu51x3g7jXgsyPLkwl5uGyL2ZTse+04lj5p3MhJpBBBq/rh0oM/YAbiNuMNjQh1ooKSpYBLCURLaP7djS+4e7N+ccHx0d2bKj0PV5njzg82P/WHuttX+tfWSIxWIxEARBEARBEARBpDEZV7oABEEQBEEQBEEQiaCJC0EQBEEQBEEQaU0sFkPWlS4EQRAEQRAEQRCEGrFYDLFYDLOzszRxIQiCIAiCIAgi/YhGo5idncVE4CzOP92e3MTl//vZv+KFnlM48/oAAr99H+s/Z0K+sRA33XwLttz5NWy584+XqtwEQRAEQRAEQfweIN1lOd/1DD5qP4xYNAqDnq+K/bTzaTz7/I/wxhu/wdjFaeQbczF7OYaZmRhycrMQM0Rxy6abkLViJep3N+H2zV9ajjoRBEEQBEEQBPEZgk1aZmZmMNJxFKNPH+X3Ek5c/vHY4+j416OYQRSXZ4Dx0RiyMrNQaDYiMjYJQ8YKRMbHcf0frMGaNeuQlS3gv3xlO+6t/daSV4wgCIIgCIIgiM8OsVgMly9fxkc/8+B82w9k9zRDxR4/8Ffo/Nm/4POfX4m3fn0BGbEoxLEYgsEJYDaErMwMrBAE/OeKzUDsPWRGP4RhOhf/9m8nEYtGcd+f/LclrRhBEARBEARBEJ8NYrEYotEoJgJn8VH7Ydm9vJKb4k9cuk89g75XjkOcmMbrr08hR8hCaPQywu9P4AvXmBCNxpCRmYkbrjUDFy4hDGBlXhQZOXmYicXQ9x8v4PqiG/DlL1UudR0JgiAIgiAIgriKkYaIfdhxFLFolN/LK7kJ6584Ej9U7I7KTVhluoSJiRgmxBnMRqMIDF/CH5nN+OKNa7EqPx+ZWZnIQAyrb7DjpcG3MLn6EgyZuQAMuM5chKz8dXAfbFmu+hIEQRAEQRAEcRXCDuNHzvlw9sGdsnul/9iFzJVG9R+g/NdnjmM2+gkuXZiBuSADMzMzyMwExi9OIz8nE7HoLAwGANEYZqMxTE9cgDlfwMefjGFycgI52QLOX4zgtyO/xcv9vUkXXBRFNDY2wm63839er3dBQtCLx+OR5cf+eTyeec81Nzfzv5ubm1WfU0MURRw8eBC7du1CQ0MDzp49CwAIh8NoamrCrl27sGvXLpw5c0bz3V27duHkyZP83pkzZ/j1pqYmhMPhhYqBIAiCIAiCIJYVttsSjUZx4dWXZPeKdn4bK661wP94o3qo2L/9+ylMjF1GZnYm3h4SkZu/AnkrDViRDUzNzGLo/CVcePc8brm+EK//9hNkZ/tw51fK8atLI4hERABjEPJWYkXmFAbf7Mcd5cmHixmNRvT09MBms8Hv9+OJJ56A3W6HyWRakEASUVtbi9raWoTDYTz++OPYv3+/al61tbX8//1+P8bGxjA4OAhBEBLm8U//9E/YsGED9u3bhzNnzuDpp5/Ggw8+iMnJSTQ2NsJiseDs2bP4x3/8R5SUlMjyDwaDuOuuu7BhwwaIoogf/vCHWLduHW677TaEQiG0trZCEAScOXMGP/7xj/Hggw/qKhNBEARBEARBXGnYjsvEr96QXS+4/SuIDJ1BZOj/qu+4vDk4hOnpDGQYZnEpMg1DZhSTEcB+jQn5OZnIMwDW1flYnZ+PCnsRbr9hLVZnzKLAYMTl6EpMTk4jOjOJ2Owshn/9q0VXpKioCEVFRQiFQotO60oRDocRDodxxx13AAA+//nPIzc3F8FgEBaLBRaLBQBgsVhgNpvn7ZqUlJRgw4YNAABBEGSTmi1btvBJislkQm5u7nJUiSAIgiAIgiBSAttxmX7/nOy6ULweF37x7wCgPnH57XsBRCZEZGYC6//QiI8+EPHRyDhsq/KRk5mJP7SsQX52Js59OIqLk5cxNjGJi6OjWGUUkBlbiQuXMjA+ngNDZgY+Hj2/6IqMjIwAmJvAKEO1/H4/6urqEAwGZeFl7JlwOIzGxkZ0d3cvKOzM4/Hg7//+71FbW4vm5mZ4PB54PB74/X7s3r0bTz31FLZt2wa/36+ZTjgcRm5uLp9UCIIAQRDmTVCCwSCAuQmMKIr4m7/5Gx5SxhBFEaIoqu4IhcNhnjZBEARBEARBpDvSH5yMRS7Ouy+eHQYQZ+JiMGQgOzMboYuziFycxapVOZi6DBjyVmBsZgaBUBgTMzEUGPOQAeATcQofXboEk2kVNm/4A/ynm2/EDddeg5mZy5iNXl5QBSKRCKqrq2G329Ha2oqHH34YgiCgqqoKwWCQD/gHBgawdetWWCwWHD58GD6fD4ODgxgbG+OTiUAggLfeegs+nw/Hjx9HR0cHRFHUXZaXXnoJbW1taGpq4tdsNhva29tx3333obu7GzabbUH1ZJw8eRK7du3C008/je985zuaE48XX3wRgiCgpKREdj0cDuP555/HV7/61UWVhSAIgiAIgiCWA/adMDZ50UJ14lK4Zg0uT8cwPTGLy7FZFP/hClxvy8PbUxcRnJlGZGYWhasFrMxegYKVAlYbc/HBzDTOhUbwzm/fwocXRpC3MgP2G80oueHaBVWCnXHxer2IRCJ818VkMsFiscDn80EURQwNDaGsrAwA4PV6YbfbsXHjRjz11FMYHR0FAFitVuzZswcAYLfbAQCTk5O6y+J0OpfsbA3jrrvuwpNPPol77rkH+/fvx9mzZyEIAv7yL/+ST1DYAX0A+M53viN7/8yZM/j+97+PxsbGeRMagiAIgiAIgkh3DAYDDMbV864LJesBxPkByttv/0/o6upEVnY2JseAj4OXcSk8gxW5GYhl5eCTiWlEcQklpgJEY5mYjEVRcK0A+x9VYuvWe7D+po2AwYCRD97Be+/9elEVMJlMqKurw4kTJ/iOx44dO/DCCy+gsLAQ1113HWw2G7xeLzo6OvhBeWk4WTpgMpkwOTmJyclJCIIQN9yrpKQEX/7yl/HWW2/JJiCiKOLHP/4x7rnnnnkTkzNnzuC1115LuzoTBEEQBEEQhB4MBgMMBgOyim7A9MVPj1KI54ZRcHslPvrpU+o7Ll+pqEY0GkNsNoqV+dkwXyMgcjGK0PkZFK+7GWuNufhgTETfuQ/Qd/Z9CDnZuO9P/wf+srEZn9/wR8jKWoGszGxYraW4445atSySorS0FMFgkJ9NKSoqwtjYGF544QXcdtttAObCwSwWCz83cvr06UXnm0pMJhNMJhNefvllAMCvfvUrTE5OwmKx4O233+aha+FwGK+99hq+8IUvyM64/OpXv4LNZps3aRFFEa+99hruueeeZa8TQRAEQRAEQaQKg8GA3Js3yq5d+MW/w1j6RzCW/pH6jsuf/Ml9+LvWJ/DB+x/g0oVpTE9Hsfbaa/CVij9GzqUIZsSPAUMGZjGL1cIK/Jc/vhtf+updAIB3330X58/PHchft24dbrzxxkVXQhAEOJ1OuN1utLW1wWQyobi4GP39/fjTP/1TAEBVVRXq6upgt9uxadMmWK3WReebau655x58//vfx/PPP4/CwkL81V/9FQRBwOTkJBoaGvhzDQ0NKCkpkZ3D+eCDD/D888/j+eef59ccDgfuuece+P1+fO973+PXBUHA9773PQoZIwiCIAiCINIag8GAWCwGg8GAjIwM5G+uwKXnjvH7I8f/AYV33gXb/v8NQyzOKZjnT3bgfz76V7juumuw6aZbYbnGimjOKsSmp/DJK/+KyzNTmI3OIi9nBb79g3Zc/4eb8O6772L16tVYtWoVAODSpUu4ePEiPve5z6W8kuwHH6W/q0IQBEEQBEEQxNUF+xTyzMwMRFHE+z98DFOv/Qe/n1dyE9Y/0aYeKgYAX7/LiZbDT2Lnf92D0ptvhbByFWKxGFaar8XazduQn29EpiEDBkMmLJ/7PADg/fffx6pVq/Dyyy/j5ZdfxqpVq/D++++nvHLhcBinTp3ih/IJgiAIgiAIgri6MRgMyMrKgunub8GQ8ek0ZeLsrzH8cF38iQsAlH+lEsU334yxyWnk5ORg1ar8uc+UGddCuPk/w7SuBMYCM2AwLHlFGB6PBw6HA3V1dYv+BDFBEARBEARBEFceFiqWmZmJPGsJ8u+Vf0F34uyv44eKSTk77MPgmTdwaXwckUtjyMowIG/1asxMjCE3Jxt3bN2K6//gevzmN7/BypUrZaFi4+PjKTnnQhAEQRAEQRDEZxMWLjY7O4vp6WlMTk7i42f/CRMnj/NndE1cGP7hd/HBewGMfjKKC+MTMESjyF6RhVvKNmHDprkvALz77rs8POzGG2+kSQtBEARBEARBEJqwH6BkZ12mp6cxNTWF0AvPY7zjHxCLRpObuBAEQRAEQRAEQSwFbPIyOzuLmZkZXL58GVNTU5gInMXFnx6niQtBEARBEARBEOkB23VhOy9sAjM9Pa3+Oy4EQRAEQRAEQRBXgozffVEsKyuLH9rPysqiiQtBEARBEARBEOkB+0HKjIwMGAwGPnHJyMigUDGCIAiCIAiCINIPduaF/dP8HZcrzWzoPN7b+Xmc3X4tpt5984qUYeL0Kfi+lIGRpj9GdHJ8SfJIVE+/34/KykrY7fZ5/zweD8LhMOrq6uD3+5ekfKlCFEU0NjbC6/Um9TyrZyrLwORXWVnJ5RZPzsryer1eNDY2QhRFfq25uRm1tbUIh8Pz8mxubkZzc3NKyq+GNH1p2RLJe6nL9VmByVGpg36/H9/4xjfm2V1zczO3y9raWi5/j8fD5b3cNpuOPkLNjq5W/H4/6urqVO1fL0zPtGTi8XhkPutKk0ivPB6Pbn+kF2mayZYnWVKpo8ttg0ttX1J/RnzKUvaramMUab+kdn8520jLD8brL/Wg3HFJ64kLMYfNZkNvby98Ph+OHz+OmpoaDA4Owufzoba29koXb8no6uqCxWJJWT3D4TC+9a1vwel0wufzwefzob29HRMTE/wZNlFh93t6euByuTSN3+PxIBgM4p//+Z9hMpnm3W9qakJTU9Oiy8/qoOz8Fpp+Ksv1WUYQBJSXl+PcuXOy6wMDAxgcHMTAwAC/JooigsEgrFYrTCYT/8Hcq4l0nOT8vmA0GvHee+9haGho3r1wOIyOjo4rUKqFU1tbi8OHD0MQhLROM93RM0lgCybE4khGjqnsj/UiHaN4vV50dHTIyqu8f/r06bTQC5vNhmeffZb/cHyyiz1s8mIwGGjiQqQv586dQ3FxccrSC4VCMJlMsNvt/JrNZkNpaWncd2w2G5577jkMDw+rDuSY49i/f//vVUf6+4bVasXp06e5kxVFEf39/fjyl78sm9CMjIwgEonIdIwg9BKJRFBQUIC+vr5591566SVYrVbSLYIgAAAmkwkul2veopr0vtPpjHv/auSqnbhceOZH8H0pg/+bOH2K35t6902c3X4tD+8K/dMB+L6UgY9+8OcAPg39ivc+C9vyfSkD7+38PGZGg/Pylz6jFkYWr3zRyXGMNP1x3LwBYHb0Q552suFpw8PDfJtQGbbEwlbUthcZalv5ym1P6cqPVtgVe/bRRx9FY2MjKisrcfbsWVl+bFtTWRaW7pEjR7Bv3z5Zuh6PJ+42Kdsaf/TRR1VDvMxmM8LhMHw+n26ZAnPGv379etnKOjD3Y6ttbW1oa2tT3WmRykEqw+bmZl5+rZAHZXv5/X7cfffd6O3tRXV1NX93oVv20veY7I4dO6Zri5m9K20P6fPx2sLr9WpuY0vvs/Iow6tYGaXX4+m2tHxSm4h3PR52ux0mkwmhUAgAMDk5CQD49re/jeHhYf7+wMAA1q9fD5PJtODQGLVVPGnYB7vv9Xp5vZW2p7RN5Uq9UmYs7Xg6pldmyvAUZXiAWt2eeeaZuPoQL0+WTnd3d1yfp1YupT7GS18pv0T+9NVXX+X31FZskwkfqa2tnbdQIooihoaG4HQ65z2/kDpoyU+5EqoMeVTT63h9T6JwJam96w2BSzYESqvPUJYhkd9ThgRrlT+RDarVS62czc3N2LdvH44cOaLar2n1l8DC7CtR+Wpra/Hxxx/L7ittQi09ad/H7iv7L6V+sfvsXel7etpUKg8tvY8nx2R9pbI+WnJJ1o+pEQgEdD+rhtI+mNz1lE3LDyqR+n6v14vq6mr09vbC4XAkPX656iYuH/3gz/Hx3z4ou/bh//wWpt59E1PvvokP/mILZi9+AgAY/8XPMNq2X/bsdODX89Jk78+GzuP9/+crmH7vnbln33sH57+/S/bs1Ltv4r1vlfJnWD6ftDQlLN8nLU0Y/8XP4tZt9uIn+OB723ja47/4GS51tmvKgxEOh/Hiiy+iu7sbg4ODsFqtOHr0KL9XV1cHl8vFtw9PnTo1r5NgITFstS8cDuP06dOyleZz586hoqICoijikUce4aFcPp8PbrcbDz30kEyx+/r60NDQgN7eXpSUlPDrfr8fu3fvhtvtnhcGJggCDh8+jPr6ehw8eBC9vb2w2WzweDzo6OiYt00qdeSdnZ3Yvn07fD7fvBAdk8mEQ4cOweVyJe0ciouLZasW7733Hh577DE8/PDDmpMWJV6vF8FgkIf6PfDAA6rPHT16lIe0DQ4O4qabbuK7P5WVlejp6Ul5uERnZyfy8vJ4iFxXV5fmoPvIkSMAwMsYDAZlHYiyLbxeL1wuF3p6elTf8Xq9cLvdvH0bGhrw5JNPyvIMh8M4f/48fD4fmpqaNHXb7/fj1KlTPD232w0Aca9roZy8+nw+WCwWlJaWoqioiE9oUr1LGI9wOAy32422trZ5tsdss7y8nNvm2NiYbMLu8/lkMgsEAhgaGoqrYx6PB/39/VxvnU4n9y9S7HY7IpEIRkZGAMwPpwuFQigqKkJRUREAbZ1LlKeWz1NDqY9a6Y+MjGDLli1cT61WK1566SWer1TnBgcH8cYbb3B/UlFRgf7+fj6wDofDGB4exo4dO3S1bWFhIbZu3YoTJ07wa0NDQ4hEIrj++utlzy60DlryKyoqgtFo5Pri8/nw5ptv8n6BTdrZzk+y7cBQ2rvb7UZra2tKz2Qk6jMS+SQpzc3NspBgrfLrsUEl77zzDk/r+PHj6OjoQDgcRlNTEw4ePIj6+nrVfi1efwkszr6kKOt66NAhPP300/w+swlpGLbT6cTjjz/O25MNTtn9vXv36m1GHDlyBBUVFfD5fNi8eTOXgVJWamVV6lU8fY0nx2R9pRQ9clmo/bB3Ozo6UFFRoXrf7/fjJz/5SVzfI4oiJiYmuA4cPHgQHR0dusqWyA9q4XA40NPTg8rKSni93qRD666qicts6DzEgV5krl4D6/87APsrUVzzFz/E7MVPIA70QRzow+zFT7Dy9q/B9vMIbD+PYOXtX5OlUfDN78L+ShT2V6L8/uzFTzA7+iGmfG9g+r13sOKGm1FyMsjTl8LyWP3178D+ShQlJ4NYccPNEAd6MX32Lc3yMVj57K9Ekbd5qyz9df+7G/ZXolj99e8AgGyCpIXJZEJDQwMEQYAgCHA6nQgGgxBFET6fD1arlYdEmUwmbN26dd4OAgCUlZVhbGwMoigiFAph8+bN2LBhA0KhEMLhMILBIOx2Ow+J2bNnD3+3tLQUVqtV5py3b9/OnShjdHQUDz30ENxut+74fxaa43K5+ESBbYNKwypqamoShn719vbC5XItaKYvrQMAVRkmIhAI8M6/tLQ07uSDTZQEQdCsU6qoqanB9u3bAczJafv27ZqrOfX19XzSyXROOmBTtkVfXx/27t3L9UH5Tl9fH5xOJ29fm802r3MzmUwyJ5xIt0OhEJ9UlJSU8LTjXddCOiDt6+tDcXExBEFAfn4+BgYG+AC1rKwsYVqLhU3CWbmltsdss6qqij+/Z88eWYiRw+HgtmcymbB58+a4bc1sz+l0cl2tqqpCMBic10mxCR6zj48//hj79+/nujwwMCDT+Xg6pydPLZ+nhlQfE6Vvs9mwbds2APPPOCl1ThAENDQ0yNoCAD+n4vP5YDQa+WRND2VlZXwnTxRFdHR0yMq62DpoyQ8AysvLuT4EAgHs37+f9wts0i71w8m0A0Np76WlpTAajXzSu1j09BmJfBLjmWeeQTAYxIEDB7istcqvxwaV3H///Twt5Q7vQlmMfUlJ5JuZTbC8WHpsEcPv92N4eFg2XqisrNS96FdfX8/9VUVFBTZt2sRlq5RVIr1KVl+T8ZVKEsllIeVhk1e73Q6Hw8HHMmr3W1tb8S//8i/zxmAMQRBw//33cx0oKytDJBLh45NkxpVKP7iUXFUTl5nQh5i98AlmL36CwJ+WwfelDNnuBhvk5zm+iozclcjIXYk8x1dlaUjDuPx3GmU7IGw3RiirRKZ5LQBghfUm2fssj4vP/xi+L2Xg7F0Wfm02/JFm+VZ//dvIXL0G47/4Gfx3Gud9RSxz9RpkFl43l+8NNy9OWBICgQA6OzuxceNGvqW3b98+1bhHs9nMHe/AwAAqKipQWlqKgYEBWYc1OjoKo9GI3Nxc/q4gCLBYLAmN+rvf/S6cTmdSh5YnJycRiURQWFgou261WnV1kkocDofm6poS5Ur6rbfeir/7u7+bdzBOT77M0WiFRrAVCK1wsnSjsLAQRqNR9Z700LrynUgkggsXLqjeT4SWbttsNrjdbuzevVu2BR7vup76RSIRhMNhjI2N8QkKm9C8//77st2E5YTZHgBV21SiDGNhu2dqTE5OIhAIYOfOnfx5h8OB119/XXVgVVxcjL6+Pi6nO++8E8FgECMjIxgaGtI1sUs2z2TRk740rGXfvn383UAgAIvFEnfBQblzzQZSyeyO2mw2rF+/Hi+99FLcc1OLqUMirFYr+vv7EQ6Hce7cOdx5550A5nZx+vr64q7w6oX5g3379vHybdy4EU899RSf9C6WRH1GKBTS9Els8NbZ2YmTJ0/KzjEmKr8eG1Qi/SKUw+FIOqQ5GZKxr3i+W4qaTeTm5sJoNC5YHvEoLCzEhg0bVNNaCr1KxlcqSSSXhcD6LLaDoxxHsfuDg4MA5j5ypIU0rK66ulp3JEoiP7iUpPUPUEp3QLLM12Em9CGAuQH+ur99ETmf2yR7np9j8b6AVTW7+f8zZkPncfH5H2Pl7V/DdY8/jdjEmCw0jCEO9GI2dB6Z5rUYe/mnqmVb/fXv4Nr//n/k5f3dJCRe+QCgpOsjRCfH8eH+e+ZC2f7hEVzT+HfJiCVprFYr6uvrdW3HsRXTV199FefPn0dVVRUKCwtx7NgxTExM8MG71LkrFTfR4PNHP/oR2tvnQuD0fi1MauzK1YOFGk+8r0UpUVstAj5d9d69ezesVqvuiRjrlPx+P1wuF9xut+qKCPs6icfjwSOPPIIDBw7or9wVYHR0FJFIRPWedFKrlFNRURFycnIAYN79RG2TSLfZDpt0W9vhcMS9rgULofn5z3/O/wY+tYVXXnkF+fn5V8SRsw5bWh6pbUp3mFgYi9PpxOHDhwFAc+cxNzcXGzZswKFDh+Ku3EkpKyvDqVOn8Mtf/hLFxcUoKiqCxWLBm2++iUgkArPZnDCNRHku5rPDetJvbm5GcXExHzx6PB6ui2xQL4oil+/o6KisTEwGZ8+e5bvUybJjxw4cO3YMQ0ND2Lp1K0wmkyyPxdQhESzk75e//CXy8/NRVFSE0tJSvPrqqxgbG5s3GUgW5g+OHz++ZF/dS9RnmM1mTZ/EBsY1NTW45ZZbUFdXx88zJiq/3+/XtEG155944gk899xzvJ1dLlcqxKBKMjYdz3dLdUnNJoC5r+QxXYk3XkglidolWb+RrK9UokcuSwXbAXG5XCgrK1NtZxY6OTg4CEEQuB7qQY8fXCrScseFHWD/4Hu/2+b+3Q5Izuc2If8rtbIdDekh9vw7/isA8B0N5Y4Kg+94SHZL5vKpQObqNZh+7x2cvcsC35cycPH5H8veZbsmbMdFesg+Ufk++sGfz9vpyVprRUZO3lKJEsBcJ3T69GndK8sVFRX46U9/ivz8fJhMJj5A6+vr46ulbBAnjcVkn95L1EkXFhaira0tqd0KNslwu92yg21ut1v36p/f70d3dzf/m22Xa51JYIfI6urqVLdA2eq9y+XSdbC0u7ubP2c2m1UHcaIo4tixY3yXJdldiOWiq6tLdujR7XZrrixXVFSgpaWFvyOKIlpbW1FaWgqz2Yzy8nJZrLLf70+4WqSl216vl1/Pzc3lcox3PdF35pkOPv7447IJCttl+fnPf77olWhWJqPRyMPdmGylhMNh2RkIJicmy3A4LDvPcOLECb7Cp1yJZmfZ4sFCFfWeP2Dy8Hg8sl0pj8cjCzHSItk8k0UrfeUKM/MTjMLCQrz++us8FIyFcklhOyYHDhxAeXk5rzP7oIKezr2oqAiRSARvv/22LOQoFXVIBFvA8ng8uO222wDMTcbYLlIqdhUrKipk/lxKKn6DRE+foeWTpH7sm9/8JpxOp+yjBVrlT2SDSpQ7Emw1falI1r6Ki4s1fbPdbkcgEJBd6+rqQiQS4f5AOV5gC0fKLzZ2dXWhs7NzwXXTapdkSdZXKkkkl6XGZrPh3nvvjdvOyl0TFvKsBz1+cKlIy4mLFOXOxrX//f/w8x9K8jZvlZ1JWf3178iezTSvlf1t+V/PyM7A5HxuE677H//M/155+9dg+V/PyPLI+dwmrPvbF5G5eo1qGbTKp2Tl7V/Dmr1L/+NA0kPper7gYrfbYTAYeIfF4vhXr17NjU0QBBw4cICvJtrtdn5YWM/AhJWppaVFdwdVW1vLQ8zixXdqYTab0d7eLttCLi8vl+36SOND2ZdgBgcHNfNwOBzYu3cvqqurE04OCwsLUV1dzctfV1c3byVEEAScP3+ehz+53W4epsDOcCi/+HQlqKioQGtrK69LovA/h8MBt9vN66+Uf21tLT94yeJzd+3aFTc9QFu3CwsL+XWWl8PhiHtdD2wgKJ2gsEGAwWBY0Mq6ElsW5Z8AACAASURBVLZS1tLSArvdjrq6OuzevXtevdeuXSvTU6mOsIUBdv8LX/gCL5tUh1j60smxmo7V1tbCYrHIQvLirTwyeUj9RWFhIfx+f1ITu2TyXAjx0mex3CyMZtu2bbIQSJvNhvb2dq5D27Ztw5YtW+b5vYqKCvj9/gWfeWLl2Lx5c1yfutA66KGiogIXL17kemM2m3Hx4kXNc3nJIPUZyi82BQIBlJeXLzqfRH1GIp+kTIv5J6/Xq1n+RDaohJ0TYO3I3mNUVVXxRcF4fcyOHTvQ0tKi++tsydiXUo5K36xW3/7+fn4mSG28cPLkSeTm5sLhcMj8/sTEBGpqahKWPx5a7aIHqRxDoVDSvlJKIrksB+x8TUtLy7x7Ur2y2+0YGhrSfUZFrx/Uen/9+vULOmtsiMVisaTeuIqQfmXsmr/4IQq++d0rXSSC+EzAwk6W+gcsWbjLZ/mHVpOFhZE8/PDDukK3iCsDC8NYzkHKZwFRFNHS0oI9e/Ysy0FfgiCuLtL6jEuyKD9nzFhxw80wVt1zhUpFEMRC8Hq96Orq4uehCOJqgYUkuVwumrQkiSAIS74gQhDE1ctnauKihtoheoIg0g+v14udO3fyv9etW4f29nbaVSCuKpqbm3HkyBEcPHhwyQ6eEwRB/L7ymQ4VIwiCIAiCIAjis0HaH84nCIIgCIIgCIKgiQtBEARBEARBEGkPTVwIgiAIgiAIgkh7aOJCEARBEARBEETaQxMXgiAIgiAIgiDSHpq4EARBEARBEASR9qTlxEUURTQ2NsLr9aYkvXA4jLq6Ovj9/gWn4fV60djYCFEUEQ6HUVtby8vn8XjQ3Ny86HJ6PB6ex1I8r5XOYsqfqvoz/H4/vvGNbyTVXl6vF3a7HbW1tQiHwykrS6pIVVslS6ptKdVI7SqdSNdyAXO/E5JKe7tSMN202+3z/jU2NiIUCmnqrtIPJ0NzczPsdvtnQo5qKH2o1Edfad2+0vnrJVG/tth6NDc3w+PxaKa/HH1aOvcRqR5bpIrF+J50IV1lq4fP/A9QLgUmk0nT4SyU2tpa1NbW8r+bm5tRXFwsu6b1/GcFm82GZ599Vvfz4XAYbW1t6OnpSZsfK1S23We1rYC5QdITTzwBt9sNk8l0pYvzmSIcDsPlcuHhhx/mur3UvyqeyO+kCkEQcPjwYRw+fFi1nokGhAv1w16vF8FgEIODg4v+VXuPx4Nz586l3S+9J+tDU0m6yuRqIh37tN9nlH3cUo0BCX2k5Y4LQSRDKBQCAJjN5itcEoIg0p1AIACLxbLoSQtBLBXUpxFEfK6aiQvbNmX/lFtcie6rPcu2+VjYAAtRSLTSp7W1yu5J0/F4PDx9rW1ftvXMQiSOHDmCffv2obKyUjVsShm+VldXh+7ublRWVqrmpQzNUKurcvtQra5SWdfW1uLjjz+WpcG2Udkz0pUJPbJQhvaxMknbiaXp9XpRXV2N3t5eOBwOWThEPH1g6R87dozfY7Jk15jMpekodUpaF6ZPTF7KtlOGFSjbQtrG7F53dzeXYzwdUGsTpcwBYGJiQpafUneVdVG2WTydiCd/NaTtp2z7Z555Jm5bSXVJKkNlmynf1dIbtXrrDclQszWPxyMrayLb09Il9q7f78fdd9+N3t5eVFdX87pL20OP3Ut1g8lLrZ3i6a4yjcX632QZHR1VtQOlb/L7/VwGajoOzOngvn37cOTIEdkz0vIrbU3NzuOl5ff7UVdXJ5O/WpjWo48+qjt/hpovVoYNsrwWGh6tR5+UNim9H0++apw+fTpuX6TVh6iVWau/0fIBifwmS0OPXicqszSdxsZGjI2NqaaTqj5NDaWNSMsYz8605JSor9LTlyXyjfFYSJ+tZWfx2k+tPZS2mGxfw55lzySj7/HSVNNtrf6RoeZX2LV03VW6KiYuXq8XLpcLPT098Pl8GBwcRDAYlCmW1v14aTkcDlnYgM/nwwMPPLDgcoqiiEceeQQWiwWHDx+GIAjweDzo7+/n6TudThw9elQzHRZCUV9fj4MHD6K3t1fXdnE4HMaLL76I7u5uDA4Owmq18rxY2crLy+Hz+eDz+bBly5ak6+j1euF2u+H1euHz+XDo0CE8/fTTsjLU1dXB5XLB5/PB6/Xi1KlT8Pv98Pv9OHXqFH/X7XbrzvfIkSOoqKiAz+fD8ePH0dLSAr/fD4fDgZ6eHlRWVsLr9aKpqUmXPoTDYZw/fx4+n4+HNHR2diIvLw8+nw979+5FdXU1+vr64PP50NPTg1/+8pcyhzwxMcHb9eDBg+jo6ACAhG0n1RPWFm63Gw899JBsgNDe3o62tjb4fD5s374dra2tcSfV77zzDpfr8ePH0dHRIUvrscceQ0NDA6/LoUOHZBPDjo4O/r7X6+V/J0JN/mowh8nqu3fvXn5PKveenh50dXXxvH0+n0yXAoEAhoaGdL0LxNcbVu9kbZMhtbWenh60tLSgrq4ObW1t3PZeeuklAPNtj+mjtNNTK4fNZsNzzz2HyspK9PT0cJ+iVRal3SvttaGhAU8++aRqneL5nVT634Wgxw5EUURrayvcbjfXlTVr1sxLq6mpCQcPHkR9fT18Ph/vA6QycrvdPI94di6Kompaeujs7MT27dt15S9FEASUl5ejr68PwFy7nz59GqdPn+a2fu7cOVRUVCxEzBwtfWL+3el0clt2Op14/PHHk5JJZ2cnXnvtNZ6GxWJBS0uLLA+1PkStrIme1fIBifzmkSNHACChXicqh5qNP/XUU6qySWWfJsXv98PlcqG9vV3VRrTsLJGcEtlovPuJfGMikumztexMq/309nHJ9DUVFRVctwDg6NGj3KYGBwdx0003Jay7nn47Uf8IzPcrADAyMoJIJIKqqipd7bDcXBUTl76+Puzdu5cPAAVBgNPpRH9/P0RRTHifMTw8jEOHDqG9vV02mAwEApicnAQAlJaWLjiEoKWlBRaLhSu2KIro7++H0+nkaVZVVSEYDC7JYTuTyYSGhgYIgsBlEAwGIYoiH+xt376dP79t27ak69rX1wen08nPMthsNtkg1OfzwWq1orS0lJdp69atGBgYADC3Bc62wUtKSnSfiaivr+edYGlpKW699VaMjo7GLWMifTCZTNixY4fsvZqaGi6fsrIybNy4kT9TVFSEG264gecpCALuv/9+Lr+ysjJEIhGuR1owp7Bnzx5+rbS0FFarFT6fj19zuVxcPjt27NBM//777+fP2u12mEwmLmcAcLvdXB42mw1f/OIXMTAwwHVUmpfJZILT6ZQ5ssXg9/sxPDwsq29lZSXPTyp3m82G7du3IxAIAJjrxFm7m0wmbN68md9L9C4QX28Wa5tSWysqKsKtt97K7YJ1BOfOnQMwvxNg+tjX15cSH6Fl94nsVQ+p8r8LJRk7YG1vMplQUlKiK32ljEpLS2E0GjEyMrIoO49HTU0N94+J8ldSVlaGsbExiKKIUCiEzZs3Y8OGDQiFQgiHwwgGg7Db7QsuG6CtT8y/S/uRqqoqRCIR1fLGo6amRqaHO3bswPDwMMLhcMI+RIqeZ7X6jkR+s76+np/10tJrrXKo2bjD4UB9fb1ueS20T5Ny4sQJ3HvvvTwNk8mEyspKfl/LzhLJKZGNxruv5Rv1kEyfrWVnyehcPJLpa9Tan/UXgiDI/IMaevvtRP0jo6qqCsPDw3yiNTAwgPXr16ftmdW0P5wviiKCweC8VaTCwkJEIhFcuHBB8z4znnA4jO9+97s4fvy4bNLicDjgcrngcDiwbt26eZMavRw5cgQ1NTU4cOAAvzY5OYlAIICdO3fKnl23bh1CodCyKkUq4rrjtYUyn87OTnR2dsqusw7A7XZj9+7d+OCDD3D8+HHdq5SLLaNSHxJhNptxww03aMYYe71eWdtu2rRJV9qjo6MwGo3Izc3l1wRBgMViQSAQSOi01PD7/VyuwJyOaVFcXAxgTkcjkQgKCwtl961Wa8oGnmr11QtbkZPq08GDBxddpkS2mQys7axWq+r90dFRHmYgpaamBhcuXNAsx2Ji3PXY60LTSMb/LsdZEkEQcODAATzyyCPYt28f6uvrdR0OZ/Vj4XFStm/fznecFmLnetCTvxSz2YyRkRGMjIxgYGAAFRUVCAQCGBgYgNVqhcVigclkWrKvUKn1I7m5uTAajRgdHV3wQXKz2Qyj0cjziNeHqJVH77NqJOs3CwsLeTn1liOej9VLKvq0xfqCZOWkFy3fmGzfo9VnJ7KzxeqRFnrav6mpiYeZsXGklt9Mdb9tMpmwfv16DAwMoKioCENDQ7j//vuTSmM5SfsdF+mATklRUREKCgo077PBkslkwo9+9CO4XK55W2UOhwM+nw/t7e1wuVwL+mxyfX09LBYLHnnkEa40ubm52LBhA9/eZf/0hn6lEqvVylfNFkq8tmArBSwfFiYg/ccGETabDb29vbJt21SSSF8WMnhWg23Lsq3fnp4e3RNRrQ4n3uBXC/bFk+eee45vGSdadWVtJh10KEnVAeZkJ40MNmmRhqWkoiMBltc2CwsLcd9993FdYf8OHz6MgoKCJS+Hlr0mIlX+dzlgoW5s11JPrDyr3/Hjx+f5LBbGtVA711tmrfyVsAHGq6++inPnzsFut6OsrAxDQ0N45513+ILEUhGvHzEajQsemANzO/GRSITnodWHKMuj91klC/Gbo6OjvJx6y6HmY9lAWg+p6NO00kjEQuSkFy3fmMoFj0R2thg90kM4HE7Y/k1NTfD5fCgvL5eNI9VYin67oqIC/f39ePfddwHM6Va6kvYTF2BOoNJ4QRbPzMK6Et1nrF+/nk9O2IC5u7ubv2c2mxe1wtnU1CSbvLAtP62zCcuF3W5HIBBAV1cXv9bd3T2vXFarVRYz3dXVJVuFKC4ulsW3+v1+WZp2ux2nT59WnZCww9zAnOEtZJCuB736sBiUK48DAwO6VzmLiopgNBpl5ym8Xi8/rJosyh0N5nSlsLh8aV5VVVU8rMntdvPyh8NhuN1uvjqXSCcWUt/e3t6E8lKuKrGY/lSwnLZZVFSESCQis5PlKAdrWy171cNi/a/X613y36IIh8M4duwY/zuZAXxFRYVM/6Uka+dms5mHOwFz8mZnNxaSf7znf/rTnyI/Px8mk4kPMPr6+lBWVqYrjYWi1o90dXUhEokkNdDp7Ozk4ctMX7Zu3QqTyaTZh6iVR++zSvT4za6uLq7XzC9KQ370lIMNmqU+eGhoKCkfmoo+TZlGOBxGb29vwvf0yGmhaPnGVKNlZ4vRo0SwEOd47S+KIo4dO8bvScdF8X7rR0+/nSwsdO6xxx7D9u3b0/qri1fFxMXhcMDtdqO6uhp2ux0bN25EeXk5jz1NdF+KzWaD2+3Gzp070dzcjMLCQv6ew+FAXV3dolY6Wdzutm3b4Pf7UVtbC4vFgo0bNyb91YwdO3agpaUl4Rel9GAymdDW1oaOjg5ejrfeemuecjocDmzevBkOhwN2ux0TExOoqanh92tra+F0Ovn91tZW7Nq1S5bPoUOH4HK55n29o7CwkF9nbZTqUDFWB736sFCqqqpkX8UZGhqSrcRqtR0La2Hx6Ha7HW63G21tbQtazWWhZUzHWBtLueWWW7Bt2zbY7Xa4XC4cOnSI56VsU2n4JJBYJ2w2G9avXx/3q2Jq9T158mTClUIWZ8zasa6uLqWT3cXYZjIIgoD9+/fLbE/6xSWtckhlsJAfu6utrZW1ndJe1VDqbir971KRm5uLN954g8uvv79f91keh8Mh03/pV64S2bn0vtfrhclkgsvlws6dO3XLWyt/Nex2OwwGA2677TYAc/qVn5+P1atXL/kqqVo/0t/fLwttUcpEjZqaGnR1dXF9sVgsXF+0+hC18uh9Vokev1lRUYHW1lbuF1k7JVsOtqjJ8urr60tq9zgVNqZMw+FwIC8vL+F7euS0UBL5xlSiZWeJ2i9RH5cI5ovU2l8QBJw/f57fc7vd2L9/PwBgbGws7mJEon47WdgimsFgSFn7LhWGWCwWu9KFIAiCIJaH5fqBSYIgCGI+0jDoeBMNv9+PF154AX/2Z3+2bOW6Wn489qrYcSEIgiAWj9frRVdX15KHFBEEQRDqdHV1IRAIaO5s2Gy2ZZ20+P1+/OQnP9H8Ml26kPZfFSMIgiAWhvKLWIv5ciJBEASRPB6PR/Y1s02bNi04NDzVSL/cqfzqbrpCoWIEQRAEQRAEQaQ9FCpGEARBEARBEETaQxMXgiAIgiAIgiDSHpq4EARBEARBEASR9tDEhSAIgiAIgiCItIcmLgRBEARBEARBpD00cSEIgiAIgiAIIu35zExcwuEwamtr4fV6r3RRkiIcDqOurg5+vz8l6fn9ftTV1SEcDid8VhRFNDY2LlhmyeS13DQ3N6O5uflKFyNl+P1+fOMb30iZnvw+4vV60djYCFEUVe+n2haXisXabbpzJdthOWTb3NwMj8ezZOlfDSjbOJX+Wq/+sLa22+2w2+2avoEgiPThMzNxMZlM8Hg8cDgci0onnQfjhDpqHVVTUxOampquYKlSi81mw7PPPntV/DgUQSwXi53kkL9PD66Evx4ZGcEDDzwAn8+HwcFBAHO/aE4QRHqTdaULQBAEQRAEsZxIF4EEQUB5eTnOnTt3BUtEEIQe0nLHhW3h9vb2yrZypdv3Ho8Hjz76KBobG1FZWYm3336bb/GrraJJr7GwMuUWsdfrRXV1NXp7e+FwOPjWtXJLWWtL2+/3o7KyUrXMXq+XX1feGx4e5u/V1tbKyi59r7KyUra6qKzLq6++KrunXI1MFC6z0LyUsLy7u7t5vTwejywNZT0Tydnj8fB77F2/34+7774bvb29qK6u5nXzeDz8fbWyaMm4sbERx44dk+Xf3NycMKRAWX5pHqwMx44dk9VNS95q8mT3Wf2k5dIKP1GTnVqZpXWTvqNMn+lRb29vXLko9SWZ8BhpvaTlVdqQtI2kbS6tW7ywH2XdOzo6NMujLL80vEVZV7W212uH7Hmv18vT1NINVne1vJX3lH4nnl7rbbtEddNTl0TtEM9fx7P9RPKQllPN3wPA6Oio7vImCnGSlqWxsRFjY2PzyhFPp4H5fQprCz11VD6nbH8gvq3Fy1dZf6l82L1jx46hsrKSt0miNpbaLtMf5ivVZJLIX0ufU9oZ00Xl+OAnP/kJduzYoSpDgiDSh7ScuDAee+wxNDQ0wOfzoaenB4cOHZJ1IH19fWhoaEBvby9KSkr49aKiIhiNRvh8Pn7txIkT2Lp1K0wmE3w+H1wuF3w+H7xeLwKBAIaGhuBwONDT04PKykp4vV40NTVBFEU88sgjKC8v51vKwWBQdTAkiiJaW1vhdrt52mvWrAEw5yzdbje8Xi+vjyAIAOY65hdffBHd3d0YHByE1WrF0aNHVd9zu91obW2VDQpYXQYHB/HGG28sOOwh1XlJ69XT04OWlhbU1dWhra2N1/Oll17istOSs8fjQX9/PwYHB+Hz+eB0OnH06FHYbDY899xzqKysRE9PDw4fPszlGq8siWTc0NCAJ598UiaXYDDI837ggQdU6zsyMoItW7bw8kvrx8pw/vx5+Hw+NDU1acpbD0eOHEFFRQV8Ph+OHz+OlpYW1cGt3+/HqVOnZPmoydzn82HLli1c3h0dHfwdr9fL/2Z0dnbitdde4+9aLBa0tLTwukr1xev14tSpU7pCetgAhKW7d+9eAHPt4HK50NPTI9ORhZwXUKv72NiYzGdIqaioQH9/v2xgPzw8jB07dvC6Op1OnpbT6cTjjz++4Jj5cDgMt9uNtrY23mYPPfSQqr3Fsw1Wz4mJCX7v4MGD6Ojo4As1anq9mLZLti562iGev45n+1rykKLm7xnt7e28vNu3b+d2mUx/EK9tnnrqKX4/kU77/X64XC60t7fL+pR4Nq1Eq/2B+LYWL19Wf4vFwt9R08033ngD3d3dOHz4MAAkZWvAnG/Jy8vjfWVXVxeXcSJ/LcVutyMSiWBkZIRf6+vrg9PphMlkktXV7XZTKC5BXAWk9cRF6khsNhu++MUvYmBggN/fvn27qqNh2759fX0APh1klJWVAZjrsNhZGJPJhM2bNyMQCKiWYWRkBJFIBFVVVTxtp9PJ01aDpWUymVBSUgJRFNHR0QGXy8Wdpc1mQ2lpKX+uoaEBgiDw9IPBIERRnOdkS0tLYTQaMTIyAp/PB6vVytMRBAENDQ382WRJdV7SehUVFeHWW2/l6Su35rXkLIoi+vv74XQ6+aSkqqoKwWBQ9yQtGRnbbDbegTMCgQAmJye5XNQmRzabDdu2bePlV4YemEwm2Yqelrz1UF9fz/W4tLQUt956K0ZHR1WfDYVCCIVCAICSkhKYTCYMDQ0BmLMjBit/f3+/TF9NJtM8va+pqZHJaceOHRgeHkY4HJ6nLyaTCVu3bpXZrxp+vx/Dw8PYs2cPv1ZZWQmTyYS+vj7s3buX2zxrR+mEQi9KfQOAPXv2wG63qz7P6sFk5vP5YDQaUVRUxOsqlWNVVdW8AVMymEwmHDp0SKYbVqt13mAvkW0IgoD777+f3ysrK0MkEuG6rKbXC227hdRFTzsk469T4SsAyHR/x44dXGbJ9AdqZXE4HKivr+fPJNLpEydO4N577+X3TSYTKisrAajbtBKt9teytXj5svpL31HTTWmdk7U1YM63MHuy2WzYvn07b3M9/prBdPfEiRMA5o8F2GLjQw89RJMWgrhKSOuJi5Li4mLdz0o7K5/Ph/Xr13PHpNy2PnLkSNx0RkdHeSgBe37nzp180CtFEAQcOHAA/f39su3tyclJRCIRFBYWJlVfURQRDAaxb98+nvfGjRvx1FNPYXR0FIFAABaLRXUQnSxLnZcgCLBYLLBarar3teR84cIFBAIB7Ny5k99zOBx4/fXXece9UFi945ULmBtsuFwuOByOhCE70rCLffv2Jcw3nrxTic1mg9vtxu7du2WhIvHaNJ6+Wq1WVb1nmM1mGI1GnnZnZyc2btwok0eiGPLR0VEYjUbk5ubKrsdrp8LCQtlAXC/x8omHcjGEDZ4EQVCVY25uLoxGY8raktmPksnJyYS2IQ2rqa6u5gP4eHq90LZbSF30tEMy/lqPPBZDMv1BIr+fSKcvXLgQ1zfFs2k14rV/srYW7x3WnvEmk8namhZ6/LWSsrIy2YKKdCwAAEajMem+mSCIK8dVdTj/3LlzuicvJpOJb2f39fWhoqICwKchIk6nk29ja8UoFxYW4r777sPDDz+sa9AuCAIOHz6Mw4cP8xj4vXv38kFMMqs6rEM4fvx43K+lsZU5VrbR0dEFhYotZ15qaMlZFEVs2LABhw4dUpVfKsoQCARk9VYO0hwOB3w+n2ZYQXNzM4qLi/nKo8fjiTvY0yPvVGKz2dDb2ysLA7JarfPaFJAPupV11Jq8hkIhRCIRAHOTnPr6+qS/FCSdiEjzkQ6OlPIqKipKelCklo90BVuNsrIynDp1CmfPnkUwGOQrxvHkmMoBERuwKcnNzdW0DRbiNzg4CEEQ4Pf78cQTT/D7anq90LZbSF0StUOy/jqRPBZLMv2Bmh2xuhcXFyfU6YKCgrj3AXWbVj6n1f4LsbV47wCIO5lYiK0lIpG/liINHZeOBYC5uv71X//1gstBEMTyk9Y7LtJYXK/Xi9OnT8u2mxNRUVGBjo4O2SBDuQoWDodx+vTpuGkUFRUhEono+kxiOBzGsWPH+N9sksVWa91ut+zgIws7SVQH6XtSCgsL8frrr/N0WEgag3WcLMSDxZovRV6LRUvOgiCgtLQ0qfMfemFt09HRIWsbaTm6u7v5arTZbIbZbJ6XjnIlkIWJaKEl71Ti9Xr5imxubi4vo91uRyAQmFdXAPP0lemOtNPv7OyU6UNrays/R2a323H69GnVlWCt36RhgwzpmQQ2OKuoqJCd42F5shAnq9WK06dP8zJ3dXWhs7NTVSZmsxnhcFh2BunEiROaOyQ2mw3r16/HgQMHUF5ezkNV1OTY1dWFSCTCJ1XJ2CF7hoW3sPSAT0PWGIlsQ7kbNDAwwOUTT6+12k6Jnrpp1SVROyTrr5fSVwDJ9QdsAiDtx4aGhmQ6mUinlffD4TB6e3vj2rQSrfZPxtZYvmrvsL45XujXQmwtHnr8tdo7TqcThw4dwtjYmMyG2G7e7/vv6hDE1URaT1xuueUWbNu2DXa7HS6XSxYnrQc2oJAOMljMa3V1Nex2O+rq6mROnw1O2FdmBEHA/v370dHRofllFmCuA3njjTf4M/39/Tz2tra2Fk6nk4cY7N69G3l5eQnr4HA4ZO9Jv6pjs9nQ3t4Ol8sFu92Obdu2YcuWLbyu7BxKS0sLr+vu3buXJK/FkkjOtbW1sFgssvAVtvIqbdOF/IhYbW0tNm/ezOvd2tqKXbt28fuFhYVcXxwOB+rq6uat5rLOkYWobNu2jYdNxUNL3qmksLCQt9vGjRtRXl4Oh8MBk8mEtrY2mczfeustCIIwT1+lYUWMmpoadHV18XQtFgtqa2sBfHqugeWr58tYwKfhlmyxwW634+TJk8jNzYXD4YDb7eZtwerC8nQ4HLJ2nJiYQE1NjWo+anX/whe+oBl3D8wNNP1+P4+Rj5dWf38/Dhw4wM9UJWOHLM21a9fKvsK0f/9+1VV+Lduoqqrig0q73Y6hoSFus/H0Opm201M3rbokaodE/lrN9rXkoUTp7xORTH8AzP0+ibQsfX19sjMuenRaet/hcCAvLy+uTSvRav9kbI3lq/YO+/BCvL5gobYWj0T+Wg273Q6DwRD3fCJBEFcPhlgsFrvShVAiDQ9YjjAaglDCwr7YAIKQw0JQ2OD894XlqHc4HIbL5cLDDz981R8Y/izVhUhfEvlr0kOC+OyQ1jsuBHEl8Hq96Orqkq2qEwQLg5J+MYkgiCuLHn999OjReYfyCYK4OrmqDucTxFLg9Xqxc+dO/ve6devQ3t5OMcOVhQAABghJREFUnRzBaW5uxpEjR3Dw4EHaBSaIK0gy/po9W1NTgwMHDixnMQmCWCLSMlSMIAiCIAiCIAhCCoWKEQRBEARBEASR9tDEhSAIgiAIgiCItIcmLgRBEARBEARBpD00cSEIgiAIgiAIIu2hiQtBEARBEARBEGkPTVwIgiAIgiAIgkh7aOJCEARBEARBEETac1X+AOXU1BREUcTU1BQuX76MaDSKVP0cjcFgQEZGBrKzs5GTkwNBEJCTk5OStAmCIAiCIAiCWBhX1Q9QiqKIS5cuYWZmBitXrkRubi6ys7ORkZEBg8GQkjxisRii0SguX76MyclJjI+PIysrC6tWrYIgCCnJgyAIgiAIgiCI5LhqJi6hUAiiKKKgoAArV65c1rzHx8dx4cIFCIIAs9m8rHkTBEEQBEEQBHEVTFyi0Sg+/vhjZGVlwWw2p2xnJVlisRhCoRBmZmZwzTXXICODjgcRBEEQBEEQxHKR9hOX8+fPIycnBwUFBUm89bsqRS/P/TcjG0BqJjwXLlzA1NQU1q5dm5L0CIIgCIIgCIJITFpPXEKhEGKxGAoLC5N4KwbAgMkL5xD77ZPIXWOEofCbwIpifm+xjI6OwmAwUNgYQRAEQRAEQSwTaRvvJIoiRFFMbnIQiwGxGKIzUxg/04wVo8/DMPULYOIkELuMVO26mM1mXj6CIAiCIAiCIJaetJ24XLp0CQUFBUmdaYkhBhgMmBo9ixWRQWRmrwImZ4Dxj4DYDHgI2SIxGAwoKCjApUuXUpIeQRAEQRAEQRDapOXEZWpqin/yWC/SiLfpkT7kZI4DK/KBy5OIZdgRM+TqTiscDuOBBx7Aa6+9FveZlStXYmZmBlNTU7rTJQiCIAiCIAhiYaTlD1CKopjkJ49jAGIwGAwQzw8i4+MXsCIvG4hOITaZBaywwzATBjJygcw8aJ11+eEPf4g333wTFoslYa4rV66EKIr0A5UEQRAEQRAEscSk5cRlamoKq1ev1vfw73ZaYjEDJs97MfvuEQgZ7yG2YjWiY5OYjV3Eio//Hpi8BjHhczAYa4DsP4ib3IMPPghgbgKTiNzcXFy8eFFfOQmCIAiCIAiCWDBpOXG5fPkysrOzdT8/e3kc4m9OImPkJ8gzBJCxIorZsYuIhDKRX2AAxj8AVkzCkPExkGUBMq+Z231Z5GH97OxsXL58eVFpEARBEARBEASRmLScuESjUX0/8BiLIRaLYuK3/4GY7x+wMn907tTOZQMmQ1MY+c041l5ngCk2joyxDxDLywdu/CaQvyIl3xfLyMhANBpNQUoEQRAEQRAEQWiRlhMXvcRiMcyMf4SZD16BELsEQywDiGYC0RVALAPm6wwosE8jY1U2sHI1DEIhkPVbIHoByKDfYCEIgiAIgiCIq4W0/KqY3p0MQ4YBs5MXEA37kDkzDcxmAZdzgPEciBdmIWSugGG6CJi4ERDXIiYaEA2/gdjlT5CKTyPr3hkiCIIgCIIgCGJRpOWOCzs7kpmZqfFUDLEYEL08hdnJcSDTAExmAbFsRCcyMStmIzQKZM6MY6VpAgbBgFhOJqLXVyEzcy1S8WOUyZ7FIQiCIAiCIAhiYaTlxCUnJweTk5PIzdX+7RUDAENGNqKxPMxMrUB2xgoglg1DLAvZ2TmYjY4iQ7gErDEjVlAA5KxAZkE+DJkZ0Poksl4mJyfpU8gEQRAEQRAEsQyk5cRFEAR88sknKCgo0HjKgBiiyMhZjVheMSY/+RArMlYgA9mIzmbBgEysyFiNy5MrkX1p5VxMXN4sYoKIjOgsDBkxzXkL+yyyFuPj41izZk2y1SMIgiAIgiAIIknS8oBGTk4OsrKyMD4+nuBJA7Ly12DFujswMb0W45FMTE9l4vL03MQlOi0g9NE0pkPjmI1EEZsuBHJLYcjMBwyLq/r4+DiysrJox4UgCIIgCIIgloG03HEBgFWrViEUCiEvLw8Gg/rWiMFgQGZ2LlatvwPRS+O4NPQCssfCyMqIYXY6CkMsAxkZArJyM5C9MgOxVUYY8syLnrTEYjFcuHABZjN9mYwgCIIgCIIgloP/HwDu+uwnuQdNAAAAAElFTkSuQmCC">

Reply:
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAy4AAABvCAYAAAAKX5T/AAAgAElEQVR4nO29f3hb1ZXv/ZV/xD6OlVhyIMhhBLaqwDR1gmGuUnxL7V6TxmnG3JK6gyBPuUMSu51x3g7jXgsyPLkwl5uGyL2ZTse+04lj5p3MhJpBBBq/rh0oM/YAbiNuMNjQh1ooKSpYBLCURLaP7djS+4e7N+ccHx0d2bKj0PV5njzg82P/WHuttX+tfWSIxWIxEARBEARBEARBpDEZV7oABEEQBEEQBEEQiaCJC0EQBEEQBEEQaU0sFkPWlS4EQRAEQRAEQRCEGrFYDLFYDLOzszRxIQiCIAiCIAgi/YhGo5idncVE4CzOP92e3MTl//vZv+KFnlM48/oAAr99H+s/Z0K+sRA33XwLttz5NWy584+XqtwEQRAEQRAEQfweIN1lOd/1DD5qP4xYNAqDnq+K/bTzaTz7/I/wxhu/wdjFaeQbczF7OYaZmRhycrMQM0Rxy6abkLViJep3N+H2zV9ajjoRBEEQBEEQBPEZgk1aZmZmMNJxFKNPH+X3Ek5c/vHY4+j416OYQRSXZ4Dx0RiyMrNQaDYiMjYJQ8YKRMbHcf0frMGaNeuQlS3gv3xlO+6t/daSV4wgCIIgCIIgiM8OsVgMly9fxkc/8+B82w9k9zRDxR4/8Ffo/Nm/4POfX4m3fn0BGbEoxLEYgsEJYDaErMwMrBAE/OeKzUDsPWRGP4RhOhf/9m8nEYtGcd+f/LclrRhBEARBEARBEJ8NYrEYotEoJgJn8VH7Ydm9vJKb4k9cuk89g75XjkOcmMbrr08hR8hCaPQywu9P4AvXmBCNxpCRmYkbrjUDFy4hDGBlXhQZOXmYicXQ9x8v4PqiG/DlL1UudR0JgiAIgiAIgriKkYaIfdhxFLFolN/LK7kJ6584Ej9U7I7KTVhluoSJiRgmxBnMRqMIDF/CH5nN+OKNa7EqPx+ZWZnIQAyrb7DjpcG3MLn6EgyZuQAMuM5chKz8dXAfbFmu+hIEQRAEQRAEcRXCDuNHzvlw9sGdsnul/9iFzJVG9R+g/NdnjmM2+gkuXZiBuSADMzMzyMwExi9OIz8nE7HoLAwGANEYZqMxTE9cgDlfwMefjGFycgI52QLOX4zgtyO/xcv9vUkXXBRFNDY2wm63839er3dBQtCLx+OR5cf+eTyeec81Nzfzv5ubm1WfU0MURRw8eBC7du1CQ0MDzp49CwAIh8NoamrCrl27sGvXLpw5c0bz3V27duHkyZP83pkzZ/j1pqYmhMPhhYqBIAiCIAiCIJYVttsSjUZx4dWXZPeKdn4bK661wP94o3qo2L/9+ylMjF1GZnYm3h4SkZu/AnkrDViRDUzNzGLo/CVcePc8brm+EK//9hNkZ/tw51fK8atLI4hERABjEPJWYkXmFAbf7Mcd5cmHixmNRvT09MBms8Hv9+OJJ56A3W6HyWRakEASUVtbi9raWoTDYTz++OPYv3+/al61tbX8//1+P8bGxjA4OAhBEBLm8U//9E/YsGED9u3bhzNnzuDpp5/Ggw8+iMnJSTQ2NsJiseDs2bP4x3/8R5SUlMjyDwaDuOuuu7BhwwaIoogf/vCHWLduHW677TaEQiG0trZCEAScOXMGP/7xj/Hggw/qKhNBEARBEARBXGnYjsvEr96QXS+4/SuIDJ1BZOj/qu+4vDk4hOnpDGQYZnEpMg1DZhSTEcB+jQn5OZnIMwDW1flYnZ+PCnsRbr9hLVZnzKLAYMTl6EpMTk4jOjOJ2Owshn/9q0VXpKioCEVFRQiFQotO60oRDocRDodxxx13AAA+//nPIzc3F8FgEBaLBRaLBQBgsVhgNpvn7ZqUlJRgw4YNAABBEGSTmi1btvBJislkQm5u7nJUiSAIgiAIgiBSAttxmX7/nOy6ULweF37x7wCgPnH57XsBRCZEZGYC6//QiI8+EPHRyDhsq/KRk5mJP7SsQX52Js59OIqLk5cxNjGJi6OjWGUUkBlbiQuXMjA+ngNDZgY+Hj2/6IqMjIwAmJvAKEO1/H4/6urqEAwGZeFl7JlwOIzGxkZ0d3cvKOzM4/Hg7//+71FbW4vm5mZ4PB54PB74/X7s3r0bTz31FLZt2wa/36+ZTjgcRm5uLp9UCIIAQRDmTVCCwSCAuQmMKIr4m7/5Gx5SxhBFEaIoqu4IhcNhnjZBEARBEARBpDvSH5yMRS7Ouy+eHQYQZ+JiMGQgOzMboYuziFycxapVOZi6DBjyVmBsZgaBUBgTMzEUGPOQAeATcQofXboEk2kVNm/4A/ynm2/EDddeg5mZy5iNXl5QBSKRCKqrq2G329Ha2oqHH34YgiCgqqoKwWCQD/gHBgawdetWWCwWHD58GD6fD4ODgxgbG+OTiUAggLfeegs+nw/Hjx9HR0cHRFHUXZaXXnoJbW1taGpq4tdsNhva29tx3333obu7GzabbUH1ZJw8eRK7du3C008/je985zuaE48XX3wRgiCgpKREdj0cDuP555/HV7/61UWVhSAIgiAIgiCWA/adMDZ50UJ14lK4Zg0uT8cwPTGLy7FZFP/hClxvy8PbUxcRnJlGZGYWhasFrMxegYKVAlYbc/HBzDTOhUbwzm/fwocXRpC3MgP2G80oueHaBVWCnXHxer2IRCJ818VkMsFiscDn80EURQwNDaGsrAwA4PV6YbfbsXHjRjz11FMYHR0FAFitVuzZswcAYLfbAQCTk5O6y+J0OpfsbA3jrrvuwpNPPol77rkH+/fvx9mzZyEIAv7yL/+ST1DYAX0A+M53viN7/8yZM/j+97+PxsbGeRMagiAIgiAIgkh3DAYDDMbV864LJesBxPkByttv/0/o6upEVnY2JseAj4OXcSk8gxW5GYhl5eCTiWlEcQklpgJEY5mYjEVRcK0A+x9VYuvWe7D+po2AwYCRD97Be+/9elEVMJlMqKurw4kTJ/iOx44dO/DCCy+gsLAQ1113HWw2G7xeLzo6OvhBeWk4WTpgMpkwOTmJyclJCIIQN9yrpKQEX/7yl/HWW2/JJiCiKOLHP/4x7rnnnnkTkzNnzuC1115LuzoTBEEQBEEQhB4MBgMMBgOyim7A9MVPj1KI54ZRcHslPvrpU+o7Ll+pqEY0GkNsNoqV+dkwXyMgcjGK0PkZFK+7GWuNufhgTETfuQ/Qd/Z9CDnZuO9P/wf+srEZn9/wR8jKWoGszGxYraW4445atSySorS0FMFgkJ9NKSoqwtjYGF544QXcdtttAObCwSwWCz83cvr06UXnm0pMJhNMJhNefvllAMCvfvUrTE5OwmKx4O233+aha+FwGK+99hq+8IUvyM64/OpXv4LNZps3aRFFEa+99hruueeeZa8TQRAEQRAEQaQKg8GA3Js3yq5d+MW/w1j6RzCW/pH6jsuf/Ml9+LvWJ/DB+x/g0oVpTE9Hsfbaa/CVij9GzqUIZsSPAUMGZjGL1cIK/Jc/vhtf+updAIB3330X58/PHchft24dbrzxxkVXQhAEOJ1OuN1utLW1wWQyobi4GP39/fjTP/1TAEBVVRXq6upgt9uxadMmWK3WReebau655x58//vfx/PPP4/CwkL81V/9FQRBwOTkJBoaGvhzDQ0NKCkpkZ3D+eCDD/D888/j+eef59ccDgfuuece+P1+fO973+PXBUHA9773PQoZIwiCIAiCINIag8GAWCwGg8GAjIwM5G+uwKXnjvH7I8f/AYV33gXb/v8NQyzOKZjnT3bgfz76V7juumuw6aZbYbnGimjOKsSmp/DJK/+KyzNTmI3OIi9nBb79g3Zc/4eb8O6772L16tVYtWoVAODSpUu4ePEiPve5z6W8kuwHH6W/q0IQBEEQBEEQxNUF+xTyzMwMRFHE+z98DFOv/Qe/n1dyE9Y/0aYeKgYAX7/LiZbDT2Lnf92D0ptvhbByFWKxGFaar8XazduQn29EpiEDBkMmLJ/7PADg/fffx6pVq/Dyyy/j5ZdfxqpVq/D++++nvHLhcBinTp3ih/IJgiAIgiAIgri6MRgMyMrKgunub8GQ8ek0ZeLsrzH8cF38iQsAlH+lEsU334yxyWnk5ORg1ar8uc+UGddCuPk/w7SuBMYCM2AwLHlFGB6PBw6HA3V1dYv+BDFBEARBEARBEFceFiqWmZmJPGsJ8u+Vf0F34uyv44eKSTk77MPgmTdwaXwckUtjyMowIG/1asxMjCE3Jxt3bN2K6//gevzmN7/BypUrZaFi4+PjKTnnQhAEQRAEQRDEZxMWLjY7O4vp6WlMTk7i42f/CRMnj/NndE1cGP7hd/HBewGMfjKKC+MTMESjyF6RhVvKNmHDprkvALz77rs8POzGG2+kSQtBEARBEARBEJqwH6BkZ12mp6cxNTWF0AvPY7zjHxCLRpObuBAEQRAEQRAEQSwFbPIyOzuLmZkZXL58GVNTU5gInMXFnx6niQtBEARBEARBEOkB23VhOy9sAjM9Pa3+Oy4EQRAEQRAEQRBXgozffVEsKyuLH9rPysqiiQtBEARBEARBEOkB+0HKjIwMGAwGPnHJyMigUDGCIAiCIAiCINIPduaF/dP8HZcrzWzoPN7b+Xmc3X4tpt5984qUYeL0Kfi+lIGRpj9GdHJ8SfJIVE+/34/KykrY7fZ5/zweD8LhMOrq6uD3+5ekfKlCFEU0NjbC6/Um9TyrZyrLwORXWVnJ5RZPzsryer1eNDY2QhRFfq25uRm1tbUIh8Pz8mxubkZzc3NKyq+GNH1p2RLJe6nL9VmByVGpg36/H9/4xjfm2V1zczO3y9raWi5/j8fD5b3cNpuOPkLNjq5W/H4/6urqVO1fL0zPtGTi8XhkPutKk0ivPB6Pbn+kF2mayZYnWVKpo8ttg0ttX1J/RnzKUvaramMUab+kdn8520jLD8brL/Wg3HFJ64kLMYfNZkNvby98Ph+OHz+OmpoaDA4Owufzoba29koXb8no6uqCxWJJWT3D4TC+9a1vwel0wufzwefzob29HRMTE/wZNlFh93t6euByuTSN3+PxIBgM4p//+Z9hMpnm3W9qakJTU9Oiy8/qoOz8Fpp+Ksv1WUYQBJSXl+PcuXOy6wMDAxgcHMTAwAC/JooigsEgrFYrTCYT/8Hcq4l0nOT8vmA0GvHee+9haGho3r1wOIyOjo4rUKqFU1tbi8OHD0MQhLROM93RM0lgCybE4khGjqnsj/UiHaN4vV50dHTIyqu8f/r06bTQC5vNhmeffZb/cHyyiz1s8mIwGGjiQqQv586dQ3FxccrSC4VCMJlMsNvt/JrNZkNpaWncd2w2G5577jkMDw+rDuSY49i/f//vVUf6+4bVasXp06e5kxVFEf39/fjyl78sm9CMjIwgEonIdIwg9BKJRFBQUIC+vr5591566SVYrVbSLYIgAAAmkwkul2veopr0vtPpjHv/auSqnbhceOZH8H0pg/+bOH2K35t6902c3X4tD+8K/dMB+L6UgY9+8OcAPg39ivc+C9vyfSkD7+38PGZGg/Pylz6jFkYWr3zRyXGMNP1x3LwBYHb0Q552suFpw8PDfJtQGbbEwlbUthcZalv5ym1P6cqPVtgVe/bRRx9FY2MjKisrcfbsWVl+bFtTWRaW7pEjR7Bv3z5Zuh6PJ+42Kdsaf/TRR1VDvMxmM8LhMHw+n26ZAnPGv379etnKOjD3Y6ttbW1oa2tT3WmRykEqw+bmZl5+rZAHZXv5/X7cfffd6O3tRXV1NX93oVv20veY7I4dO6Zri5m9K20P6fPx2sLr9WpuY0vvs/Iow6tYGaXX4+m2tHxSm4h3PR52ux0mkwmhUAgAMDk5CQD49re/jeHhYf7+wMAA1q9fD5PJtODQGLVVPGnYB7vv9Xp5vZW2p7RN5Uq9UmYs7Xg6pldmyvAUZXiAWt2eeeaZuPoQL0+WTnd3d1yfp1YupT7GS18pv0T+9NVXX+X31FZskwkfqa2tnbdQIooihoaG4HQ65z2/kDpoyU+5EqoMeVTT63h9T6JwJam96w2BSzYESqvPUJYhkd9ThgRrlT+RDarVS62czc3N2LdvH44cOaLar2n1l8DC7CtR+Wpra/Hxxx/L7ittQi09ad/H7iv7L6V+sfvsXel7etpUKg8tvY8nx2R9pbI+WnJJ1o+pEQgEdD+rhtI+mNz1lE3LDyqR+n6v14vq6mr09vbC4XAkPX656iYuH/3gz/Hx3z4ou/bh//wWpt59E1PvvokP/mILZi9+AgAY/8XPMNq2X/bsdODX89Jk78+GzuP9/+crmH7vnbln33sH57+/S/bs1Ltv4r1vlfJnWD6ftDQlLN8nLU0Y/8XP4tZt9uIn+OB723ja47/4GS51tmvKgxEOh/Hiiy+iu7sbg4ODsFqtOHr0KL9XV1cHl8vFtw9PnTo1r5NgITFstS8cDuP06dOyleZz586hoqICoijikUce4aFcPp8PbrcbDz30kEyx+/r60NDQgN7eXpSUlPDrfr8fu3fvhtvtnhcGJggCDh8+jPr6ehw8eBC9vb2w2WzweDzo6OiYt00qdeSdnZ3Yvn07fD7fvBAdk8mEQ4cOweVyJe0ciouLZasW7733Hh577DE8/PDDmpMWJV6vF8FgkIf6PfDAA6rPHT16lIe0DQ4O4qabbuK7P5WVlejp6Ul5uERnZyfy8vJ4iFxXV5fmoPvIkSMAwMsYDAZlHYiyLbxeL1wuF3p6elTf8Xq9cLvdvH0bGhrw5JNPyvIMh8M4f/48fD4fmpqaNHXb7/fj1KlTPD232w0Aca9roZy8+nw+WCwWlJaWoqioiE9oUr1LGI9wOAy32422trZ5tsdss7y8nNvm2NiYbMLu8/lkMgsEAhgaGoqrYx6PB/39/VxvnU4n9y9S7HY7IpEIRkZGAMwPpwuFQigqKkJRUREAbZ1LlKeWz1NDqY9a6Y+MjGDLli1cT61WK1566SWer1TnBgcH8cYbb3B/UlFRgf7+fj6wDofDGB4exo4dO3S1bWFhIbZu3YoTJ07wa0NDQ4hEIrj++utlzy60DlryKyoqgtFo5Pri8/nw5ptv8n6BTdrZzk+y7cBQ2rvb7UZra2tKz2Qk6jMS+SQpzc3NspBgrfLrsUEl77zzDk/r+PHj6OjoQDgcRlNTEw4ePIj6+nrVfi1efwkszr6kKOt66NAhPP300/w+swlpGLbT6cTjjz/O25MNTtn9vXv36m1GHDlyBBUVFfD5fNi8eTOXgVJWamVV6lU8fY0nx2R9pRQ9clmo/bB3Ozo6UFFRoXrf7/fjJz/5SVzfI4oiJiYmuA4cPHgQHR0dusqWyA9q4XA40NPTg8rKSni93qRD666qicts6DzEgV5krl4D6/87APsrUVzzFz/E7MVPIA70QRzow+zFT7Dy9q/B9vMIbD+PYOXtX5OlUfDN78L+ShT2V6L8/uzFTzA7+iGmfG9g+r13sOKGm1FyMsjTl8LyWP3178D+ShQlJ4NYccPNEAd6MX32Lc3yMVj57K9Ekbd5qyz9df+7G/ZXolj99e8AgGyCpIXJZEJDQwMEQYAgCHA6nQgGgxBFET6fD1arlYdEmUwmbN26dd4OAgCUlZVhbGwMoigiFAph8+bN2LBhA0KhEMLhMILBIOx2Ow+J2bNnD3+3tLQUVqtV5py3b9/OnShjdHQUDz30ENxut+74fxaa43K5+ESBbYNKwypqamoShn719vbC5XItaKYvrQMAVRkmIhAI8M6/tLQ07uSDTZQEQdCsU6qoqanB9u3bAczJafv27ZqrOfX19XzSyXROOmBTtkVfXx/27t3L9UH5Tl9fH5xOJ29fm802r3MzmUwyJ5xIt0OhEJ9UlJSU8LTjXddCOiDt6+tDcXExBEFAfn4+BgYG+AC1rKwsYVqLhU3CWbmltsdss6qqij+/Z88eWYiRw+HgtmcymbB58+a4bc1sz+l0cl2tqqpCMBic10mxCR6zj48//hj79+/nujwwMCDT+Xg6pydPLZ+nhlQfE6Vvs9mwbds2APPPOCl1ThAENDQ0yNoCAD+n4vP5YDQa+WRND2VlZXwnTxRFdHR0yMq62DpoyQ8AysvLuT4EAgHs37+f9wts0i71w8m0A0Np76WlpTAajXzSu1j09BmJfBLjmWeeQTAYxIEDB7istcqvxwaV3H///Twt5Q7vQlmMfUlJ5JuZTbC8WHpsEcPv92N4eFg2XqisrNS96FdfX8/9VUVFBTZt2sRlq5RVIr1KVl+T8ZVKEsllIeVhk1e73Q6Hw8HHMmr3W1tb8S//8i/zxmAMQRBw//33cx0oKytDJBLh45NkxpVKP7iUXFUTl5nQh5i98AlmL36CwJ+WwfelDNnuBhvk5zm+iozclcjIXYk8x1dlaUjDuPx3GmU7IGw3RiirRKZ5LQBghfUm2fssj4vP/xi+L2Xg7F0Wfm02/JFm+VZ//dvIXL0G47/4Gfx3Gud9RSxz9RpkFl43l+8NNy9OWBICgQA6OzuxceNGvqW3b98+1bhHs9nMHe/AwAAqKipQWlqKgYEBWYc1OjoKo9GI3Nxc/q4gCLBYLAmN+rvf/S6cTmdSh5YnJycRiURQWFgou261WnV1kkocDofm6poS5Ur6rbfeir/7u7+bdzBOT77M0WiFRrAVCK1wsnSjsLAQRqNR9Z700LrynUgkggsXLqjeT4SWbttsNrjdbuzevVu2BR7vup76RSIRhMNhjI2N8QkKm9C8//77st2E5YTZHgBV21SiDGNhu2dqTE5OIhAIYOfOnfx5h8OB119/XXVgVVxcjL6+Pi6nO++8E8FgECMjIxgaGtI1sUs2z2TRk740rGXfvn383UAgAIvFEnfBQblzzQZSyeyO2mw2rF+/Hi+99FLcc1OLqUMirFYr+vv7EQ6Hce7cOdx5550A5nZx+vr64q7w6oX5g3379vHybdy4EU899RSf9C6WRH1GKBTS9Els8NbZ2YmTJ0/KzjEmKr8eG1Qi/SKUw+FIOqQ5GZKxr3i+W4qaTeTm5sJoNC5YHvEoLCzEhg0bVNNaCr1KxlcqSSSXhcD6LLaDoxxHsfuDg4MA5j5ypIU0rK66ulp3JEoiP7iUpPUPUEp3QLLM12Em9CGAuQH+ur99ETmf2yR7np9j8b6AVTW7+f8zZkPncfH5H2Pl7V/DdY8/jdjEmCw0jCEO9GI2dB6Z5rUYe/mnqmVb/fXv4Nr//n/k5f3dJCRe+QCgpOsjRCfH8eH+e+ZC2f7hEVzT+HfJiCVprFYr6uvrdW3HsRXTV199FefPn0dVVRUKCwtx7NgxTExM8MG71LkrFTfR4PNHP/oR2tvnQuD0fi1MauzK1YOFGk+8r0UpUVstAj5d9d69ezesVqvuiRjrlPx+P1wuF9xut+qKCPs6icfjwSOPPIIDBw7or9wVYHR0FJFIRPWedFKrlFNRURFycnIAYN79RG2TSLfZDpt0W9vhcMS9rgULofn5z3/O/wY+tYVXXnkF+fn5V8SRsw5bWh6pbUp3mFgYi9PpxOHDhwFAc+cxNzcXGzZswKFDh+Ku3EkpKyvDqVOn8Mtf/hLFxcUoKiqCxWLBm2++iUgkArPZnDCNRHku5rPDetJvbm5GcXExHzx6PB6ui2xQL4oil+/o6KisTEwGZ8+e5bvUybJjxw4cO3YMQ0ND2Lp1K0wmkyyPxdQhESzk75e//CXy8/NRVFSE0tJSvPrqqxgbG5s3GUgW5g+OHz++ZF/dS9RnmM1mTZ/EBsY1NTW45ZZbUFdXx88zJiq/3+/XtEG155944gk899xzvJ1dLlcqxKBKMjYdz3dLdUnNJoC5r+QxXYk3XkglidolWb+RrK9UokcuSwXbAXG5XCgrK1NtZxY6OTg4CEEQuB7qQY8fXCrScseFHWD/4Hu/2+b+3Q5Izuc2If8rtbIdDekh9vw7/isA8B0N5Y4Kg+94SHZL5vKpQObqNZh+7x2cvcsC35cycPH5H8veZbsmbMdFesg+Ufk++sGfz9vpyVprRUZO3lKJEsBcJ3T69GndK8sVFRX46U9/ivz8fJhMJj5A6+vr46ulbBAnjcVkn95L1EkXFhaira0tqd0KNslwu92yg21ut1v36p/f70d3dzf/m22Xa51JYIfI6urqVLdA2eq9y+XSdbC0u7ubP2c2m1UHcaIo4tixY3yXJdldiOWiq6tLdujR7XZrrixXVFSgpaWFvyOKIlpbW1FaWgqz2Yzy8nJZrLLf70+4WqSl216vl1/Pzc3lcox3PdF35pkOPv7447IJCttl+fnPf77olWhWJqPRyMPdmGylhMNh2RkIJicmy3A4LDvPcOLECb7Cp1yJZmfZ4sFCFfWeP2Dy8Hg8sl0pj8cjCzHSItk8k0UrfeUKM/MTjMLCQrz++us8FIyFcklhOyYHDhxAeXk5rzP7oIKezr2oqAiRSARvv/22LOQoFXVIBFvA8ng8uO222wDMTcbYLlIqdhUrKipk/lxKKn6DRE+foeWTpH7sm9/8JpxOp+yjBVrlT2SDSpQ7Emw1falI1r6Ki4s1fbPdbkcgEJBd6+rqQiQS4f5AOV5gC0fKLzZ2dXWhs7NzwXXTapdkSdZXKkkkl6XGZrPh3nvvjdvOyl0TFvKsBz1+cKlIy4mLFOXOxrX//f/w8x9K8jZvlZ1JWf3178iezTSvlf1t+V/PyM7A5HxuE677H//M/155+9dg+V/PyPLI+dwmrPvbF5G5eo1qGbTKp2Tl7V/Dmr1L/+NA0kPper7gYrfbYTAYeIfF4vhXr17NjU0QBBw4cICvJtrtdn5YWM/AhJWppaVFdwdVW1vLQ8zixXdqYTab0d7eLttCLi8vl+36SOND2ZdgBgcHNfNwOBzYu3cvqqurE04OCwsLUV1dzctfV1c3byVEEAScP3+ehz+53W4epsDOcCi/+HQlqKioQGtrK69LovA/h8MBt9vN66+Uf21tLT94yeJzd+3aFTc9QFu3CwsL+XWWl8PhiHtdD2wgKJ2gsEGAwWBY0Mq6ElsW5Z8AACAASURBVLZS1tLSArvdjrq6OuzevXtevdeuXSvTU6mOsIUBdv8LX/gCL5tUh1j60smxmo7V1tbCYrHIQvLirTwyeUj9RWFhIfx+f1ITu2TyXAjx0mex3CyMZtu2bbIQSJvNhvb2dq5D27Ztw5YtW+b5vYqKCvj9/gWfeWLl2Lx5c1yfutA66KGiogIXL17kemM2m3Hx4kXNc3nJIPUZyi82BQIBlJeXLzqfRH1GIp+kTIv5J6/Xq1n+RDaohJ0TYO3I3mNUVVXxRcF4fcyOHTvQ0tKi++tsydiXUo5K36xW3/7+fn4mSG28cPLkSeTm5sLhcMj8/sTEBGpqahKWPx5a7aIHqRxDoVDSvlJKIrksB+x8TUtLy7x7Ur2y2+0YGhrSfUZFrx/Uen/9+vULOmtsiMVisaTeuIqQfmXsmr/4IQq++d0rXSSC+EzAwk6W+gcsWbjLZ/mHVpOFhZE8/PDDukK3iCsDC8NYzkHKZwFRFNHS0oI9e/Ysy0FfgiCuLtL6jEuyKD9nzFhxw80wVt1zhUpFEMRC8Hq96Orq4uehCOJqgYUkuVwumrQkiSAIS74gQhDE1ctnauKihtoheoIg0g+v14udO3fyv9etW4f29nbaVSCuKpqbm3HkyBEcPHhwyQ6eEwRB/L7ymQ4VIwiCIAiCIAjis0HaH84nCIIgCIIgCIKgiQtBEARBEARBEGkPTVwIgiAIgiAIgkh7aOJCEARBEARBEETaQxMXgiAIgiAIgiDSHpq4EARBEARBEASR9qTlxEUURTQ2NsLr9aYkvXA4jLq6Ovj9/gWn4fV60djYCFEUEQ6HUVtby8vn8XjQ3Ny86HJ6PB6ex1I8r5XOYsqfqvoz/H4/vvGNbyTVXl6vF3a7HbW1tQiHwykrS6pIVVslS6ptKdVI7SqdSNdyAXO/E5JKe7tSMN202+3z/jU2NiIUCmnqrtIPJ0NzczPsdvtnQo5qKH2o1Edfad2+0vnrJVG/tth6NDc3w+PxaKa/HH1aOvcRqR5bpIrF+J50IV1lq4fP/A9QLgUmk0nT4SyU2tpa1NbW8r+bm5tRXFwsu6b1/GcFm82GZ599Vvfz4XAYbW1t6OnpSZsfK1S23We1rYC5QdITTzwBt9sNk8l0pYvzmSIcDsPlcuHhhx/mur3UvyqeyO+kCkEQcPjwYRw+fFi1nokGhAv1w16vF8FgEIODg4v+VXuPx4Nz586l3S+9J+tDU0m6yuRqIh37tN9nlH3cUo0BCX2k5Y4LQSRDKBQCAJjN5itcEoIg0p1AIACLxbLoSQtBLBXUpxFEfK6aiQvbNmX/lFtcie6rPcu2+VjYAAtRSLTSp7W1yu5J0/F4PDx9rW1ftvXMQiSOHDmCffv2obKyUjVsShm+VldXh+7ublRWVqrmpQzNUKurcvtQra5SWdfW1uLjjz+WpcG2Udkz0pUJPbJQhvaxMknbiaXp9XpRXV2N3t5eOBwOWThEPH1g6R87dozfY7Jk15jMpekodUpaF6ZPTF7KtlOGFSjbQtrG7F53dzeXYzwdUGsTpcwBYGJiQpafUneVdVG2WTydiCd/NaTtp2z7Z555Jm5bSXVJKkNlmynf1dIbtXrrDclQszWPxyMrayLb09Il9q7f78fdd9+N3t5eVFdX87pL20OP3Ut1g8lLrZ3i6a4yjcX632QZHR1VtQOlb/L7/VwGajoOzOngvn37cOTIEdkz0vIrbU3NzuOl5ff7UVdXJ5O/WpjWo48+qjt/hpovVoYNsrwWGh6tR5+UNim9H0++apw+fTpuX6TVh6iVWau/0fIBifwmS0OPXicqszSdxsZGjI2NqaaTqj5NDaWNSMsYz8605JSor9LTlyXyjfFYSJ+tZWfx2k+tPZS2mGxfw55lzySj7/HSVNNtrf6RoeZX2LV03VW6KiYuXq8XLpcLPT098Pl8GBwcRDAYlCmW1v14aTkcDlnYgM/nwwMPPLDgcoqiiEceeQQWiwWHDx+GIAjweDzo7+/n6TudThw9elQzHRZCUV9fj4MHD6K3t1fXdnE4HMaLL76I7u5uDA4Owmq18rxY2crLy+Hz+eDz+bBly5ak6+j1euF2u+H1euHz+XDo0CE8/fTTsjLU1dXB5XLB5/PB6/Xi1KlT8Pv98Pv9OHXqFH/X7XbrzvfIkSOoqKiAz+fD8ePH0dLSAr/fD4fDgZ6eHlRWVsLr9aKpqUmXPoTDYZw/fx4+n4+HNHR2diIvLw8+nw979+5FdXU1+vr64PP50NPTg1/+8pcyhzwxMcHb9eDBg+jo6ACAhG0n1RPWFm63Gw899JBsgNDe3o62tjb4fD5s374dra2tcSfV77zzDpfr8ePH0dHRIUvrscceQ0NDA6/LoUOHZBPDjo4O/r7X6+V/J0JN/mowh8nqu3fvXn5PKveenh50dXXxvH0+n0yXAoEAhoaGdL0LxNcbVu9kbZMhtbWenh60tLSgrq4ObW1t3PZeeuklAPNtj+mjtNNTK4fNZsNzzz2HyspK9PT0cJ+iVRal3SvttaGhAU8++aRqneL5nVT634Wgxw5EUURrayvcbjfXlTVr1sxLq6mpCQcPHkR9fT18Ph/vA6QycrvdPI94di6Kompaeujs7MT27dt15S9FEASUl5ejr68PwFy7nz59GqdPn+a2fu7cOVRUVCxEzBwtfWL+3el0clt2Op14/PHHk5JJZ2cnXnvtNZ6GxWJBS0uLLA+1PkStrIme1fIBifzmkSNHACChXicqh5qNP/XUU6qySWWfJsXv98PlcqG9vV3VRrTsLJGcEtlovPuJfGMikumztexMq/309nHJ9DUVFRVctwDg6NGj3KYGBwdx0003Jay7nn47Uf8IzPcrADAyMoJIJIKqqipd7bDcXBUTl76+Puzdu5cPAAVBgNPpRH9/P0RRTHifMTw8jEOHDqG9vV02mAwEApicnAQAlJaWLjiEoKWlBRaLhSu2KIro7++H0+nkaVZVVSEYDC7JYTuTyYSGhgYIgsBlEAwGIYoiH+xt376dP79t27ak69rX1wen08nPMthsNtkg1OfzwWq1orS0lJdp69atGBgYADC3Bc62wUtKSnSfiaivr+edYGlpKW699VaMjo7GLWMifTCZTNixY4fsvZqaGi6fsrIybNy4kT9TVFSEG264gecpCALuv/9+Lr+ysjJEIhGuR1owp7Bnzx5+rbS0FFarFT6fj19zuVxcPjt27NBM//777+fP2u12mEwmLmcAcLvdXB42mw1f/OIXMTAwwHVUmpfJZILT6ZQ5ssXg9/sxPDwsq29lZSXPTyp3m82G7du3IxAIAJjrxFm7m0wmbN68md9L9C4QX28Wa5tSWysqKsKtt97K7YJ1BOfOnQMwvxNg+tjX15cSH6Fl94nsVQ+p8r8LJRk7YG1vMplQUlKiK32ljEpLS2E0GjEyMrIoO49HTU0N94+J8ldSVlaGsbExiKKIUCiEzZs3Y8OGDQiFQgiHwwgGg7Db7QsuG6CtT8y/S/uRqqoqRCIR1fLGo6amRqaHO3bswPDwMMLhcMI+RIqeZ7X6jkR+s76+np/10tJrrXKo2bjD4UB9fb1ueS20T5Ny4sQJ3HvvvTwNk8mEyspKfl/LzhLJKZGNxruv5Rv1kEyfrWVnyehcPJLpa9Tan/UXgiDI/IMaevvtRP0jo6qqCsPDw3yiNTAwgPXr16ftmdW0P5wviiKCweC8VaTCwkJEIhFcuHBB8z4znnA4jO9+97s4fvy4bNLicDjgcrngcDiwbt26eZMavRw5cgQ1NTU4cOAAvzY5OYlAIICdO3fKnl23bh1CodCyKkUq4rrjtYUyn87OTnR2dsqusw7A7XZj9+7d+OCDD3D8+HHdq5SLLaNSHxJhNptxww03aMYYe71eWdtu2rRJV9qjo6MwGo3Izc3l1wRBgMViQSAQSOi01PD7/VyuwJyOaVFcXAxgTkcjkQgKCwtl961Wa8oGnmr11QtbkZPq08GDBxddpkS2mQys7axWq+r90dFRHmYgpaamBhcuXNAsx2Ji3PXY60LTSMb/LsdZEkEQcODAATzyyCPYt28f6uvrdR0OZ/Vj4XFStm/fznecFmLnetCTvxSz2YyRkRGMjIxgYGAAFRUVCAQCGBgYgNVqhcVigclkWrKvUKn1I7m5uTAajRgdHV3wQXKz2Qyj0cjziNeHqJVH77NqJOs3CwsLeTn1liOej9VLKvq0xfqCZOWkFy3fmGzfo9VnJ7KzxeqRFnrav6mpiYeZsXGklt9Mdb9tMpmwfv16DAwMoKioCENDQ7j//vuTSmM5SfsdF+mATklRUREKCgo077PBkslkwo9+9CO4XK55W2UOhwM+nw/t7e1wuVwL+mxyfX09LBYLHnnkEa40ubm52LBhA9/eZf/0hn6lEqvVylfNFkq8tmArBSwfFiYg/ccGETabDb29vbJt21SSSF8WMnhWg23Lsq3fnp4e3RNRrQ4n3uBXC/bFk+eee45vGSdadWVtJh10KEnVAeZkJ40MNmmRhqWkoiMBltc2CwsLcd9993FdYf8OHz6MgoKCJS+Hlr0mIlX+dzlgoW5s11JPrDyr3/Hjx+f5LBbGtVA711tmrfyVsAHGq6++inPnzsFut6OsrAxDQ0N45513+ILEUhGvHzEajQsemANzO/GRSITnodWHKMuj91klC/Gbo6OjvJx6y6HmY9lAWg+p6NO00kjEQuSkFy3fmMoFj0R2thg90kM4HE7Y/k1NTfD5fCgvL5eNI9VYin67oqIC/f39ePfddwHM6Va6kvYTF2BOoNJ4QRbPzMK6Et1nrF+/nk9O2IC5u7ubv2c2mxe1wtnU1CSbvLAtP62zCcuF3W5HIBBAV1cXv9bd3T2vXFarVRYz3dXVJVuFKC4ulsW3+v1+WZp2ux2nT59WnZCww9zAnOEtZJCuB736sBiUK48DAwO6VzmLiopgNBpl5ym8Xi8/rJosyh0N5nSlsLh8aV5VVVU8rMntdvPyh8NhuN1uvjqXSCcWUt/e3t6E8lKuKrGY/lSwnLZZVFSESCQis5PlKAdrWy171cNi/a/X613y36IIh8M4duwY/zuZAXxFRYVM/6Uka+dms5mHOwFz8mZnNxaSf7znf/rTnyI/Px8mk4kPMPr6+lBWVqYrjYWi1o90dXUhEokkNdDp7Ozk4ctMX7Zu3QqTyaTZh6iVR++zSvT4za6uLq7XzC9KQ370lIMNmqU+eGhoKCkfmoo+TZlGOBxGb29vwvf0yGmhaPnGVKNlZ4vRo0SwEOd47S+KIo4dO8bvScdF8X7rR0+/nSwsdO6xxx7D9u3b0/qri1fFxMXhcMDtdqO6uhp2ux0bN25EeXk5jz1NdF+KzWaD2+3Gzp070dzcjMLCQv6ew+FAXV3dolY6Wdzutm3b4Pf7UVtbC4vFgo0bNyb91YwdO3agpaUl4Rel9GAymdDW1oaOjg5ejrfeemuecjocDmzevBkOhwN2ux0TExOoqanh92tra+F0Ovn91tZW7Nq1S5bPoUOH4HK55n29o7CwkF9nbZTqUDFWB736sFCqqqpkX8UZGhqSrcRqtR0La2Hx6Ha7HW63G21tbQtazWWhZUzHWBtLueWWW7Bt2zbY7Xa4XC4cOnSI56VsU2n4JJBYJ2w2G9avXx/3q2Jq9T158mTClUIWZ8zasa6uLqWT3cXYZjIIgoD9+/fLbE/6xSWtckhlsJAfu6utrZW1ndJe1VDqbir971KRm5uLN954g8uvv79f91keh8Mh03/pV64S2bn0vtfrhclkgsvlws6dO3XLWyt/Nex2OwwGA2677TYAc/qVn5+P1atXL/kqqVo/0t/fLwttUcpEjZqaGnR1dXF9sVgsXF+0+hC18uh9Vokev1lRUYHW1lbuF1k7JVsOtqjJ8urr60tq9zgVNqZMw+FwIC8vL+F7euS0UBL5xlSiZWeJ2i9RH5cI5ovU2l8QBJw/f57fc7vd2L9/PwBgbGws7mJEon47WdgimsFgSFn7LhWGWCwWu9KFIAiCIJaH5fqBSYIgCGI+0jDoeBMNv9+PF154AX/2Z3+2bOW6Wn489qrYcSEIgiAWj9frRVdX15KHFBEEQRDqdHV1IRAIaO5s2Gy2ZZ20+P1+/OQnP9H8Ml26kPZfFSMIgiAWhvKLWIv5ciJBEASRPB6PR/Y1s02bNi04NDzVSL/cqfzqbrpCoWIEQRAEQRAEQaQ9FCpGEARBEARBEETaQxMXgiAIgiAIgiDSHpq4EARBEARBEASR9tDEhSAIgiAIgiCItIcmLgRBEARBEARBpD00cSEIgiAIgiAIIu35zExcwuEwamtr4fV6r3RRkiIcDqOurg5+vz8l6fn9ftTV1SEcDid8VhRFNDY2LlhmyeS13DQ3N6O5uflKFyNl+P1+fOMb30iZnvw+4vV60djYCFEUVe+n2haXisXabbpzJdthOWTb3NwMj8ezZOlfDSjbOJX+Wq/+sLa22+2w2+2avoEgiPThMzNxMZlM8Hg8cDgci0onnQfjhDpqHVVTUxOampquYKlSi81mw7PPPntV/DgUQSwXi53kkL9PD66Evx4ZGcEDDzwAn8+HwcFBAHO/aE4QRHqTdaULQBAEQRAEsZxIF4EEQUB5eTnOnTt3BUtEEIQe0nLHhW3h9vb2yrZypdv3Ho8Hjz76KBobG1FZWYm3336bb/GrraJJr7GwMuUWsdfrRXV1NXp7e+FwOPjWtXJLWWtL2+/3o7KyUrXMXq+XX1feGx4e5u/V1tbKyi59r7KyUra6qKzLq6++KrunXI1MFC6z0LyUsLy7u7t5vTwejywNZT0Tydnj8fB77F2/34+7774bvb29qK6u5nXzeDz8fbWyaMm4sbERx44dk+Xf3NycMKRAWX5pHqwMx44dk9VNS95q8mT3Wf2k5dIKP1GTnVqZpXWTvqNMn+lRb29vXLko9SWZ8BhpvaTlVdqQtI2kbS6tW7ywH2XdOzo6NMujLL80vEVZV7W212uH7Hmv18vT1NINVne1vJX3lH4nnl7rbbtEddNTl0TtEM9fx7P9RPKQllPN3wPA6Oio7vImCnGSlqWxsRFjY2PzyhFPp4H5fQprCz11VD6nbH8gvq3Fy1dZf6l82L1jx46hsrKSt0miNpbaLtMf5ivVZJLIX0ufU9oZ00Xl+OAnP/kJduzYoSpDgiDSh7ScuDAee+wxNDQ0wOfzoaenB4cOHZJ1IH19fWhoaEBvby9KSkr49aKiIhiNRvh8Pn7txIkT2Lp1K0wmE3w+H1wuF3w+H7xeLwKBAIaGhuBwONDT04PKykp4vV40NTVBFEU88sgjKC8v51vKwWBQdTAkiiJaW1vhdrt52mvWrAEw5yzdbje8Xi+vjyAIAOY65hdffBHd3d0YHByE1WrF0aNHVd9zu91obW2VDQpYXQYHB/HGG28sOOwh1XlJ69XT04OWlhbU1dWhra2N1/Oll17istOSs8fjQX9/PwYHB+Hz+eB0OnH06FHYbDY899xzqKysRE9PDw4fPszlGq8siWTc0NCAJ598UiaXYDDI837ggQdU6zsyMoItW7bw8kvrx8pw/vx5+Hw+NDU1acpbD0eOHEFFRQV8Ph+OHz+OlpYW1cGt3+/HqVOnZPmoydzn82HLli1c3h0dHfwdr9fL/2Z0dnbitdde4+9aLBa0tLTwukr1xev14tSpU7pCetgAhKW7d+9eAHPt4HK50NPTI9ORhZwXUKv72NiYzGdIqaioQH9/v2xgPzw8jB07dvC6Op1OnpbT6cTjjz++4Jj5cDgMt9uNtrY23mYPPfSQqr3Fsw1Wz4mJCX7v4MGD6Ojo4As1anq9mLZLti562iGev45n+1rykKLm7xnt7e28vNu3b+d2mUx/EK9tnnrqKX4/kU77/X64XC60t7fL+pR4Nq1Eq/2B+LYWL19Wf4vFwt9R08033ngD3d3dOHz4MAAkZWvAnG/Jy8vjfWVXVxeXcSJ/LcVutyMSiWBkZIRf6+vrg9PphMlkktXV7XZTKC5BXAWk9cRF6khsNhu++MUvYmBggN/fvn27qqNh2759fX0APh1klJWVAZjrsNhZGJPJhM2bNyMQCKiWYWRkBJFIBFVVVTxtp9PJ01aDpWUymVBSUgJRFNHR0QGXy8Wdpc1mQ2lpKX+uoaEBgiDw9IPBIERRnOdkS0tLYTQaMTIyAp/PB6vVytMRBAENDQ382WRJdV7SehUVFeHWW2/l6Su35rXkLIoi+vv74XQ6+aSkqqoKwWBQ9yQtGRnbbDbegTMCgQAmJye5XNQmRzabDdu2bePlV4YemEwm2Yqelrz1UF9fz/W4tLQUt956K0ZHR1WfDYVCCIVCAICSkhKYTCYMDQ0BmLMjBit/f3+/TF9NJtM8va+pqZHJaceOHRgeHkY4HJ6nLyaTCVu3bpXZrxp+vx/Dw8PYs2cPv1ZZWQmTyYS+vj7s3buX2zxrR+mEQi9KfQOAPXv2wG63qz7P6sFk5vP5YDQaUVRUxOsqlWNVVdW8AVMymEwmHDp0SKYbVqt13mAvkW0IgoD777+f3ysrK0MkEuG6rKbXC227hdRFTzsk469T4SsAyHR/x44dXGbJ9AdqZXE4HKivr+fPJNLpEydO4N577+X3TSYTKisrAajbtBKt9teytXj5svpL31HTTWmdk7U1YM63MHuy2WzYvn07b3M9/prBdPfEiRMA5o8F2GLjQw89RJMWgrhKSOuJi5Li4mLdz0o7K5/Ph/Xr13PHpNy2PnLkSNx0RkdHeSgBe37nzp180CtFEAQcOHAA/f39su3tyclJRCIRFBYWJlVfURQRDAaxb98+nvfGjRvx1FNPYXR0FIFAABaLRXUQnSxLnZcgCLBYLLBarar3teR84cIFBAIB7Ny5k99zOBx4/fXXece9UFi945ULmBtsuFwuOByOhCE70rCLffv2Jcw3nrxTic1mg9vtxu7du2WhIvHaNJ6+Wq1WVb1nmM1mGI1GnnZnZyc2btwok0eiGPLR0VEYjUbk5ubKrsdrp8LCQtlAXC/x8omHcjGEDZ4EQVCVY25uLoxGY8raktmPksnJyYS2IQ2rqa6u5gP4eHq90LZbSF30tEMy/lqPPBZDMv1BIr+fSKcvXLgQ1zfFs2k14rV/srYW7x3WnvEmk8namhZ6/LWSsrIy2YKKdCwAAEajMem+mSCIK8dVdTj/3LlzuicvJpOJb2f39fWhoqICwKchIk6nk29ja8UoFxYW4r777sPDDz+sa9AuCAIOHz6Mw4cP8xj4vXv38kFMMqs6rEM4fvx43K+lsZU5VrbR0dEFhYotZ15qaMlZFEVs2LABhw4dUpVfKsoQCARk9VYO0hwOB3w+n2ZYQXNzM4qLi/nKo8fjiTvY0yPvVGKz2dDb2ysLA7JarfPaFJAPupV11Jq8hkIhRCIRAHOTnPr6+qS/FCSdiEjzkQ6OlPIqKipKelCklo90BVuNsrIynDp1CmfPnkUwGOQrxvHkmMoBERuwKcnNzdW0DRbiNzg4CEEQ4Pf78cQTT/D7anq90LZbSF0StUOy/jqRPBZLMv2Bmh2xuhcXFyfU6YKCgrj3AXWbVj6n1f4LsbV47wCIO5lYiK0lIpG/liINHZeOBYC5uv71X//1gstBEMTyk9Y7LtJYXK/Xi9OnT8u2mxNRUVGBjo4O2SBDuQoWDodx+vTpuGkUFRUhEono+kxiOBzGsWPH+N9sksVWa91ut+zgIws7SVQH6XtSCgsL8frrr/N0WEgag3WcLMSDxZovRV6LRUvOgiCgtLQ0qfMfemFt09HRIWsbaTm6u7v5arTZbIbZbJ6XjnIlkIWJaKEl71Ti9Xr5imxubi4vo91uRyAQmFdXAPP0lemOtNPv7OyU6UNrays/R2a323H69GnVlWCt36RhgwzpmQQ2OKuoqJCd42F5shAnq9WK06dP8zJ3dXWhs7NTVSZmsxnhcFh2BunEiROaOyQ2mw3r16/HgQMHUF5ezkNV1OTY1dWFSCTCJ1XJ2CF7hoW3sPSAT0PWGIlsQ7kbNDAwwOUTT6+12k6Jnrpp1SVROyTrr5fSVwDJ9QdsAiDtx4aGhmQ6mUinlffD4TB6e3vj2rQSrfZPxtZYvmrvsL45XujXQmwtHnr8tdo7TqcThw4dwtjYmMyG2G7e7/vv6hDE1URaT1xuueUWbNu2DXa7HS6XSxYnrQc2oJAOMljMa3V1Nex2O+rq6mROnw1O2FdmBEHA/v370dHRofllFmCuA3njjTf4M/39/Tz2tra2Fk6nk4cY7N69G3l5eQnr4HA4ZO9Jv6pjs9nQ3t4Ol8sFu92Obdu2YcuWLbyu7BxKS0sLr+vu3buXJK/FkkjOtbW1sFgssvAVtvIqbdOF/IhYbW0tNm/ezOvd2tqKXbt28fuFhYVcXxwOB+rq6uat5rLOkYWobNu2jYdNxUNL3qmksLCQt9vGjRtRXl4Oh8MBk8mEtrY2mczfeustCIIwT1+lYUWMmpoadHV18XQtFgtqa2sBfHqugeWr58tYwKfhlmyxwW634+TJk8jNzYXD4YDb7eZtwerC8nQ4HLJ2nJiYQE1NjWo+anX/whe+oBl3D8wNNP1+P4+Rj5dWf38/Dhw4wM9UJWOHLM21a9fKvsK0f/9+1VV+Lduoqqrig0q73Y6hoSFus/H0Opm201M3rbokaodE/lrN9rXkoUTp7xORTH8AzP0+ibQsfX19sjMuenRaet/hcCAvLy+uTSvRav9kbI3lq/YO+/BCvL5gobYWj0T+Wg273Q6DwRD3fCJBEFcPhlgsFrvShVAiDQ9YjjAaglDCwr7YAIKQw0JQ2OD894XlqHc4HIbL5cLDDz981R8Y/izVhUhfEvlr0kOC+OyQ1jsuBHEl8Hq96Orqkq2qEwQLg5J+MYkgiCuLHn999OjReYfyCYK4OrmqDucTxFLg9Xqxc+dO/ve6devQ3t5OMcOVhQAABghJREFUnRzBaW5uxpEjR3Dw4EHaBSaIK0gy/po9W1NTgwMHDixnMQmCWCLSMlSMIAiCIAiCIAhCCoWKEQRBEARBEASR9tDEhSAIgiAIgiCItIcmLgRBEARBEARBpD00cSEIgiAIgiAIIu2hiQtBEARBEARBEGkPTVwIgiAIgiAIgkh7aOJCEARBEARBEETac1X+AOXU1BREUcTU1BQuX76MaDSKVP0cjcFgQEZGBrKzs5GTkwNBEJCTk5OStAmCIAiCIAiCWBhX1Q9QiqKIS5cuYWZmBitXrkRubi6ys7ORkZEBg8GQkjxisRii0SguX76MyclJjI+PIysrC6tWrYIgCCnJgyAIgiAIgiCI5LhqJi6hUAiiKKKgoAArV65c1rzHx8dx4cIFCIIAs9m8rHkTBEEQBEEQBHEVTFyi0Sg+/vhjZGVlwWw2p2xnJVlisRhCoRBmZmZwzTXXICODjgcRBEEQBEEQxHKR9hOX8+fPIycnBwUFBUm89bsqRS/P/TcjG0BqJjwXLlzA1NQU1q5dm5L0CIIgCIIgCIJITFpPXEKhEGKxGAoLC5N4KwbAgMkL5xD77ZPIXWOEofCbwIpifm+xjI6OwmAwUNgYQRAEQRAEQSwTaRvvJIoiRFFMbnIQiwGxGKIzUxg/04wVo8/DMPULYOIkELuMVO26mM1mXj6CIAiCIAiCIJaetJ24XLp0CQUFBUmdaYkhBhgMmBo9ixWRQWRmrwImZ4Dxj4DYDHgI2SIxGAwoKCjApUuXUpIeQRAEQRAEQRDapOXEZWpqin/yWC/SiLfpkT7kZI4DK/KBy5OIZdgRM+TqTiscDuOBBx7Aa6+9FveZlStXYmZmBlNTU7rTJQiCIAiCIAhiYaTlD1CKopjkJ49jAGIwGAwQzw8i4+MXsCIvG4hOITaZBaywwzATBjJygcw8aJ11+eEPf4g333wTFoslYa4rV66EKIr0A5UEQRAEQRAEscSk5cRlamoKq1ev1vfw73ZaYjEDJs97MfvuEQgZ7yG2YjWiY5OYjV3Eio//Hpi8BjHhczAYa4DsP4ib3IMPPghgbgKTiNzcXFy8eFFfOQmCIAiCIAiCWDBpOXG5fPkysrOzdT8/e3kc4m9OImPkJ8gzBJCxIorZsYuIhDKRX2AAxj8AVkzCkPExkGUBMq+Z231Z5GH97OxsXL58eVFpEARBEARBEASRmLScuESjUX0/8BiLIRaLYuK3/4GY7x+wMn907tTOZQMmQ1MY+c041l5ngCk2joyxDxDLywdu/CaQvyIl3xfLyMhANBpNQUoEQRAEQRAEQWiRlhMXvcRiMcyMf4SZD16BELsEQywDiGYC0RVALAPm6wwosE8jY1U2sHI1DEIhkPVbIHoByKDfYCEIgiAIgiCIq4W0/KqY3p0MQ4YBs5MXEA37kDkzDcxmAZdzgPEciBdmIWSugGG6CJi4ERDXIiYaEA2/gdjlT5CKTyPr3hkiCIIgCIIgCGJRpOWOCzs7kpmZqfFUDLEYEL08hdnJcSDTAExmAbFsRCcyMStmIzQKZM6MY6VpAgbBgFhOJqLXVyEzcy1S8WOUyZ7FIQiCIAiCIAhiYaTlxCUnJweTk5PIzdX+7RUDAENGNqKxPMxMrUB2xgoglg1DLAvZ2TmYjY4iQ7gErDEjVlAA5KxAZkE+DJkZ0Poksl4mJyfpU8gEQRAEQRAEsQyk5cRFEAR88sknKCgo0HjKgBiiyMhZjVheMSY/+RArMlYgA9mIzmbBgEysyFiNy5MrkX1p5VxMXN4sYoKIjOgsDBkxzXkL+yyyFuPj41izZk2y1SMIgiAIgiAIIknS8oBGTk4OsrKyMD4+nuBJA7Ly12DFujswMb0W45FMTE9l4vL03MQlOi0g9NE0pkPjmI1EEZsuBHJLYcjMBwyLq/r4+DiysrJox4UgCIIgCIIgloG03HEBgFWrViEUCiEvLw8Gg/rWiMFgQGZ2LlatvwPRS+O4NPQCssfCyMqIYXY6CkMsAxkZArJyM5C9MgOxVUYY8syLnrTEYjFcuHABZjN9mYwgCIIgCIIgloP/HwDu+uwnuQdNAAAAAElFTkSuQmCC">

Reply:
resolved